{
  "nbformat": 4,
  "nbformat_minor": 0,
  "metadata": {
    "colab": {
      "provenance": []
    },
    "kernelspec": {
      "name": "python3",
      "display_name": "Python 3"
    },
    "language_info": {
      "name": "python"
    }
  },
  "cells": [
    {
      "cell_type": "code",
      "execution_count": null,
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "eey0F9Yuj4Fv",
        "outputId": "a690117d-604a-4e4d-afda-ecf97dd5fe56"
      },
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "Reading package lists... Done\n",
            "Building dependency tree... Done\n",
            "Reading state information... Done\n",
            "E: Unable to locate package ghost\n",
            "E: Unable to locate package script\n",
            "--2024-05-13 20:13:54--  http://homer.ucsd.edu/homer/configureHomer.pl\n",
            "Resolving homer.ucsd.edu (homer.ucsd.edu)... 169.228.63.226\n",
            "Connecting to homer.ucsd.edu (homer.ucsd.edu)|169.228.63.226|:80... connected.\n",
            "HTTP request sent, awaiting response... 200 OK\n",
            "Length: 27679 (27K) [application/x-perl]\n",
            "Saving to: ‘configureHomer.pl’\n",
            "\n",
            "configureHomer.pl   100%[===================>]  27.03K  --.-KB/s    in 0.1s    \n",
            "\n",
            "2024-05-13 20:13:55 (180 KB/s) - ‘configureHomer.pl’ saved [27679/27679]\n",
            "\n",
            "\n",
            "\tCurrent base directory for HOMER is /content/./\n",
            "\n",
            "--2024-05-13 20:13:55--  http://homer.ucsd.edu/homer/update.txt\n",
            "Resolving homer.ucsd.edu (homer.ucsd.edu)... 169.228.63.226\n",
            "Connecting to homer.ucsd.edu (homer.ucsd.edu)|169.228.63.226|:80... connected.\n",
            "HTTP request sent, awaiting response... 200 OK\n",
            "Length: 17857 (17K) [text/plain]\n",
            "Saving to: ‘/content/.//update.txt’\n",
            "\n",
            "/content/.//update. 100%[===================>]  17.44K   115KB/s    in 0.2s    \n",
            "\n",
            "2024-05-13 20:13:55 (115 KB/s) - ‘/content/.//update.txt’ saved [17857/17857]\n",
            "\n",
            "\tUpdating Settings...\n",
            "`wget -O 0.35307387968442.tmp http://homer.ucsd.edu/homer/configureHomer.pl`;\n",
            "--2024-05-13 20:13:55--  http://homer.ucsd.edu/homer/configureHomer.pl\n",
            "Resolving homer.ucsd.edu (homer.ucsd.edu)... 169.228.63.226\n",
            "Connecting to homer.ucsd.edu (homer.ucsd.edu)|169.228.63.226|:80... connected.\n",
            "HTTP request sent, awaiting response... 200 OK\n",
            "Length: 27679 (27K) [application/x-perl]\n",
            "Saving to: ‘0.35307387968442.tmp’\n",
            "\n",
            "0.35307387968442.tm 100%[===================>]  27.03K   178KB/s    in 0.2s    \n",
            "\n",
            "2024-05-13 20:13:56 (178 KB/s) - ‘0.35307387968442.tmp’ saved [27679/27679]\n",
            "\n",
            "\tconfigureHomer.pl script is up-to-date\n",
            "\tPackages to Install...\n",
            "\t\thomer -> \n",
            "\n",
            "\tChecking for standard utilities and 3rd party software:\n",
            "\n",
            "\t\t(Note: seqlogo, gs, and blat no longer required)\n",
            "\tChecking for wget... /usr/bin/wget\n",
            "\tChecking for cut... /usr/bin/cut\n",
            "\tChecking for gcc... /usr/bin/gcc\n",
            "\tChecking for g++... /usr/bin/g++\n",
            "\tChecking for zip... /usr/bin/zip\n",
            "\tChecking for unzip... /usr/bin/unzip\n",
            "\tChecking for make... /usr/bin/make\n",
            "\tChecking for tar... /usr/bin/tar\n",
            "\tChecking for gunzip... /usr/bin/gunzip\n",
            "\tChecking for gzip... /usr/bin/gzip\n",
            "\n",
            "\tAll auxilary programs found.\n",
            "\n",
            "\tPausing for 3 seconds (hit ctrl-C to stop)\n",
            "\tInstalling: homer\n",
            "\t\tDownloading...\n",
            "--2024-05-13 20:13:59--  http://homer.ucsd.edu/homer/data/software/homer.v5.0.1.zip\n",
            "Resolving homer.ucsd.edu (homer.ucsd.edu)... 169.228.63.226\n",
            "Connecting to homer.ucsd.edu (homer.ucsd.edu)|169.228.63.226|:80... connected.\n",
            "HTTP request sent, awaiting response... 200 OK\n",
            "Length: 38163801 (36M) [application/zip]\n",
            "Saving to: ‘homer.package.zip’\n",
            "\n",
            "homer.package.zip   100%[===================>]  36.40M  11.9MB/s    in 3.1s    \n",
            "\n",
            "2024-05-13 20:14:02 (11.9 MB/s) - ‘homer.package.zip’ saved [38163801/38163801]\n",
            "\n",
            "\t\tUnzipping...\n",
            "\t\tFinished Installing homer\n",
            "\n",
            "In file included from \u001b[01m\u001b[K/usr/include/stdio.h:894\u001b[m\u001b[K,\n",
            "                 from \u001b[01m\u001b[KMotif.h:17\u001b[m\u001b[K,\n",
            "                 from \u001b[01m\u001b[KMotif.cpp:19\u001b[m\u001b[K:\n",
            "In function ‘\u001b[01m\u001b[Kint fprintf(FILE*, const char*, ...)\u001b[m\u001b[K’,\n",
            "    inlined from ‘\u001b[01m\u001b[Kchar** findBestSeeds(CommandLine*, XMerData*, int*, int, int&)\u001b[m\u001b[K’ at \u001b[01m\u001b[KMotif.cpp:2904:11\u001b[m\u001b[K:\n",
            "\u001b[01m\u001b[K/usr/include/x86_64-linux-gnu/bits/stdio2.h:105:24:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[K‘\u001b[01m\u001b[K%s\u001b[m\u001b[K’ directive argument is null [\u001b[01;35m\u001b[K\u001b]8;;https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html#index-Wformat-overflow=\u0007-Wformat-overflow=\u001b]8;;\u0007\u001b[m\u001b[K]\n",
            "  105 |   return \u001b[01;35m\u001b[K__fprintf_chk (__stream, __USE_FORTIFY_LEVEL - 1, __fmt,\u001b[m\u001b[K\n",
            "      |          \u001b[01;35m\u001b[K~~~~~~~~~~~~~~^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~\u001b[m\u001b[K\n",
            "  106 | \u001b[01;35m\u001b[K                        __va_arg_pack ())\u001b[m\u001b[K;\n",
            "      |                         \u001b[01;35m\u001b[K~~~~~~~~~~~~~~~~~\u001b[m\u001b[K\n",
            "In function ‘\u001b[01m\u001b[Kint fprintf(FILE*, const char*, ...)\u001b[m\u001b[K’,\n",
            "    inlined from ‘\u001b[01m\u001b[Kvoid findMotifs(CommandLine*, XMerData*)\u001b[m\u001b[K’ at \u001b[01m\u001b[KMotif.cpp:177:12\u001b[m\u001b[K:\n",
            "\u001b[01m\u001b[K/usr/include/x86_64-linux-gnu/bits/stdio2.h:105:24:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[K‘\u001b[01m\u001b[K%s\u001b[m\u001b[K’ directive argument is null [\u001b[01;35m\u001b[K\u001b]8;;https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html#index-Wformat-overflow=\u0007-Wformat-overflow=\u001b]8;;\u0007\u001b[m\u001b[K]\n",
            "  105 |   return \u001b[01;35m\u001b[K__fprintf_chk (__stream, __USE_FORTIFY_LEVEL - 1, __fmt,\u001b[m\u001b[K\n",
            "      |          \u001b[01;35m\u001b[K~~~~~~~~~~~~~~^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~\u001b[m\u001b[K\n",
            "  106 | \u001b[01;35m\u001b[K                        __va_arg_pack ())\u001b[m\u001b[K;\n",
            "      |                         \u001b[01;35m\u001b[K~~~~~~~~~~~~~~~~~\u001b[m\u001b[K\n",
            "In function ‘\u001b[01m\u001b[Kint fprintf(FILE*, const char*, ...)\u001b[m\u001b[K’,\n",
            "    inlined from ‘\u001b[01m\u001b[Kvoid findMotifs(CommandLine*, XMerData*)\u001b[m\u001b[K’ at \u001b[01m\u001b[KMotif.cpp:173:12\u001b[m\u001b[K:\n",
            "\u001b[01m\u001b[K/usr/include/x86_64-linux-gnu/bits/stdio2.h:105:24:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[K‘\u001b[01m\u001b[K%s\u001b[m\u001b[K’ directive argument is null [\u001b[01;35m\u001b[K\u001b]8;;https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html#index-Wformat-overflow=\u0007-Wformat-overflow=\u001b]8;;\u0007\u001b[m\u001b[K]\n",
            "  105 |   return \u001b[01;35m\u001b[K__fprintf_chk (__stream, __USE_FORTIFY_LEVEL - 1, __fmt,\u001b[m\u001b[K\n",
            "      |          \u001b[01;35m\u001b[K~~~~~~~~~~~~~~^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~\u001b[m\u001b[K\n",
            "  106 | \u001b[01;35m\u001b[K                        __va_arg_pack ())\u001b[m\u001b[K;\n",
            "      |                         \u001b[01;35m\u001b[K~~~~~~~~~~~~~~~~~\u001b[m\u001b[K\n",
            "\u001b[01m\u001b[Khomer.cpp:\u001b[m\u001b[K In function ‘\u001b[01m\u001b[Kint main(int, char**)\u001b[m\u001b[K’:\n",
            "\u001b[01m\u001b[Khomer.cpp:71:29:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kignoring return value of ‘\u001b[01m\u001b[Kint system(const char*)\u001b[m\u001b[K’ declared with attribute ‘\u001b[01m\u001b[Kwarn_unused_result\u001b[m\u001b[K’ [\u001b[01;35m\u001b[K\u001b]8;;https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html#index-Wunused-result\u0007-Wunused-result\u001b]8;;\u0007\u001b[m\u001b[K]\n",
            "   71 |                 (void)\u001b[01;35m\u001b[Ksystem(c)\u001b[m\u001b[K;\n",
            "      |                       \u001b[01;35m\u001b[K~~~~~~^~~\u001b[m\u001b[K\n",
            "\u001b[01m\u001b[Khomer.cpp:82:37:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kignoring return value of ‘\u001b[01m\u001b[Kint system(const char*)\u001b[m\u001b[K’ declared with attribute ‘\u001b[01m\u001b[Kwarn_unused_result\u001b[m\u001b[K’ [\u001b[01;35m\u001b[K\u001b]8;;https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html#index-Wunused-result\u0007-Wunused-result\u001b]8;;\u0007\u001b[m\u001b[K]\n",
            "   82 |                         (void)\u001b[01;35m\u001b[Ksystem(c)\u001b[m\u001b[K;\n",
            "      |                               \u001b[01;35m\u001b[K~~~~~~^~~\u001b[m\u001b[K\n",
            "\u001b[01m\u001b[KSeqTag.cpp:\u001b[m\u001b[K In member function ‘\u001b[01m\u001b[Kvoid TagLibrary::makeDirectory()\u001b[m\u001b[K’:\n",
            "\u001b[01m\u001b[KSeqTag.cpp:6545:21:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kignoring return value of ‘\u001b[01m\u001b[Kint system(const char*)\u001b[m\u001b[K’ declared with attribute ‘\u001b[01m\u001b[Kwarn_unused_result\u001b[m\u001b[K’ [\u001b[01;35m\u001b[K\u001b]8;;https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html#index-Wunused-result\u0007-Wunused-result\u001b]8;;\u0007\u001b[m\u001b[K]\n",
            " 6545 |         (void)\u001b[01;35m\u001b[Ksystem(filename)\u001b[m\u001b[K;\n",
            "      |               \u001b[01;35m\u001b[K~~~~~~^~~~~~~~~~\u001b[m\u001b[K\n",
            "\u001b[01m\u001b[KSeqTag.cpp:6549:21:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kignoring return value of ‘\u001b[01m\u001b[Kint system(const char*)\u001b[m\u001b[K’ declared with attribute ‘\u001b[01m\u001b[Kwarn_unused_result\u001b[m\u001b[K’ [\u001b[01;35m\u001b[K\u001b]8;;https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html#index-Wunused-result\u0007-Wunused-result\u001b]8;;\u0007\u001b[m\u001b[K]\n",
            " 6549 |         (void)\u001b[01;35m\u001b[Ksystem(filename)\u001b[m\u001b[K;\n",
            "      |               \u001b[01;35m\u001b[K~~~~~~^~~~~~~~~~\u001b[m\u001b[K\n",
            "\u001b[01m\u001b[KSeqTag.cpp:\u001b[m\u001b[K In function ‘\u001b[01m\u001b[Kchar* unzipFileIfNeeded(char*, int&, int&)\u001b[m\u001b[K’:\n",
            "\u001b[01m\u001b[KSeqTag.cpp:6581:37:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kignoring return value of ‘\u001b[01m\u001b[Kint system(const char*)\u001b[m\u001b[K’ declared with attribute ‘\u001b[01m\u001b[Kwarn_unused_result\u001b[m\u001b[K’ [\u001b[01;35m\u001b[K\u001b]8;;https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html#index-Wunused-result\u0007-Wunused-result\u001b]8;;\u0007\u001b[m\u001b[K]\n",
            " 6581 |                         (void)\u001b[01;35m\u001b[Ksystem(command)\u001b[m\u001b[K;\n",
            "      |                               \u001b[01;35m\u001b[K~~~~~~^~~~~~~~~\u001b[m\u001b[K\n",
            "\u001b[01m\u001b[KSeqTag.cpp:6588:37:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kignoring return value of ‘\u001b[01m\u001b[Kint system(const char*)\u001b[m\u001b[K’ declared with attribute ‘\u001b[01m\u001b[Kwarn_unused_result\u001b[m\u001b[K’ [\u001b[01;35m\u001b[K\u001b]8;;https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html#index-Wunused-result\u0007-Wunused-result\u001b]8;;\u0007\u001b[m\u001b[K]\n",
            " 6588 |                         (void)\u001b[01;35m\u001b[Ksystem(command)\u001b[m\u001b[K;\n",
            "      |                               \u001b[01;35m\u001b[K~~~~~~^~~~~~~~~\u001b[m\u001b[K\n",
            "\u001b[01m\u001b[KSeqTag.cpp:6595:37:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kignoring return value of ‘\u001b[01m\u001b[Kint system(const char*)\u001b[m\u001b[K’ declared with attribute ‘\u001b[01m\u001b[Kwarn_unused_result\u001b[m\u001b[K’ [\u001b[01;35m\u001b[K\u001b]8;;https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html#index-Wunused-result\u0007-Wunused-result\u001b]8;;\u0007\u001b[m\u001b[K]\n",
            " 6595 |                         (void)\u001b[01;35m\u001b[Ksystem(command)\u001b[m\u001b[K;\n",
            "      |                               \u001b[01;35m\u001b[K~~~~~~^~~~~~~~~\u001b[m\u001b[K\n",
            "\u001b[01m\u001b[KSeqTag.cpp:6603:37:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kignoring return value of ‘\u001b[01m\u001b[Kint system(const char*)\u001b[m\u001b[K’ declared with attribute ‘\u001b[01m\u001b[Kwarn_unused_result\u001b[m\u001b[K’ [\u001b[01;35m\u001b[K\u001b]8;;https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html#index-Wunused-result\u0007-Wunused-result\u001b]8;;\u0007\u001b[m\u001b[K]\n",
            " 6603 |                         (void)\u001b[01;35m\u001b[Ksystem(command)\u001b[m\u001b[K;\n",
            "      |                               \u001b[01;35m\u001b[K~~~~~~^~~~~~~~~\u001b[m\u001b[K\n",
            "\u001b[01m\u001b[KSeqTag.cpp:\u001b[m\u001b[K In function ‘\u001b[01m\u001b[Kvoid rezipFileIfNeeded(char*, int)\u001b[m\u001b[K’:\n",
            "\u001b[01m\u001b[KSeqTag.cpp:6618:29:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kignoring return value of ‘\u001b[01m\u001b[Kint system(const char*)\u001b[m\u001b[K’ declared with attribute ‘\u001b[01m\u001b[Kwarn_unused_result\u001b[m\u001b[K’ [\u001b[01;35m\u001b[K\u001b]8;;https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html#index-Wunused-result\u0007-Wunused-result\u001b]8;;\u0007\u001b[m\u001b[K]\n",
            " 6618 |                 (void)\u001b[01;35m\u001b[Ksystem(command)\u001b[m\u001b[K;\n",
            "      |                       \u001b[01;35m\u001b[K~~~~~~^~~~~~~~~\u001b[m\u001b[K\n",
            "\u001b[01m\u001b[KSeqTag.cpp:\u001b[m\u001b[K In member function ‘\u001b[01m\u001b[Kvoid TagLibrary::readPEAlignment(char*, int, int, int)\u001b[m\u001b[K’:\n",
            "\u001b[01m\u001b[KSeqTag.cpp:6905:29:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kignoring return value of ‘\u001b[01m\u001b[Kint system(const char*)\u001b[m\u001b[K’ declared with attribute ‘\u001b[01m\u001b[Kwarn_unused_result\u001b[m\u001b[K’ [\u001b[01;35m\u001b[K\u001b]8;;https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html#index-Wunused-result\u0007-Wunused-result\u001b]8;;\u0007\u001b[m\u001b[K]\n",
            " 6905 |                 (void)\u001b[01;35m\u001b[Ksystem(fname)\u001b[m\u001b[K;\n",
            "      |                       \u001b[01;35m\u001b[K~~~~~~^~~~~~~\u001b[m\u001b[K\n",
            "\u001b[01m\u001b[KSeqTag.cpp:6907:29:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kignoring return value of ‘\u001b[01m\u001b[Kint system(const char*)\u001b[m\u001b[K’ declared with attribute ‘\u001b[01m\u001b[Kwarn_unused_result\u001b[m\u001b[K’ [\u001b[01;35m\u001b[K\u001b]8;;https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html#index-Wunused-result\u0007-Wunused-result\u001b]8;;\u0007\u001b[m\u001b[K]\n",
            " 6907 |                 (void)\u001b[01;35m\u001b[Ksystem(fname)\u001b[m\u001b[K;\n",
            "      |                       \u001b[01;35m\u001b[K~~~~~~^~~~~~~\u001b[m\u001b[K\n",
            "\u001b[01m\u001b[KSeqTag.cpp:\u001b[m\u001b[K In member function ‘\u001b[01m\u001b[Kvoid UniqMapChrs::loadData()\u001b[m\u001b[K’:\n",
            "\u001b[01m\u001b[KSeqTag.cpp:18548:20:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kignoring return value of ‘\u001b[01m\u001b[Ksize_t fread(void*, size_t, size_t, FILE*)\u001b[m\u001b[K’ declared with attribute ‘\u001b[01m\u001b[Kwarn_unused_result\u001b[m\u001b[K’ [\u001b[01;35m\u001b[K\u001b]8;;https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html#index-Wunused-result\u0007-Wunused-result\u001b]8;;\u0007\u001b[m\u001b[K]\n",
            "18548 |         (void)\u001b[01;35m\u001b[Kfread(&psize,sizeof(unsigned int),1,fpp)\u001b[m\u001b[K;\n",
            "      |               \u001b[01;35m\u001b[K~~~~~^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~\u001b[m\u001b[K\n",
            "\u001b[01m\u001b[KSeqTag.cpp:18550:20:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kignoring return value of ‘\u001b[01m\u001b[Ksize_t fread(void*, size_t, size_t, FILE*)\u001b[m\u001b[K’ declared with attribute ‘\u001b[01m\u001b[Kwarn_unused_result\u001b[m\u001b[K’ [\u001b[01;35m\u001b[K\u001b]8;;https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html#index-Wunused-result\u0007-Wunused-result\u001b]8;;\u0007\u001b[m\u001b[K]\n",
            "18550 |         (void)\u001b[01;35m\u001b[Kfread(&nsize,sizeof(unsigned int),1,fpn)\u001b[m\u001b[K;\n",
            "      |               \u001b[01;35m\u001b[K~~~~~^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~\u001b[m\u001b[K\n",
            "\u001b[01m\u001b[KSeqTag.cpp:18560:20:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kignoring return value of ‘\u001b[01m\u001b[Ksize_t fread(void*, size_t, size_t, FILE*)\u001b[m\u001b[K’ declared with attribute ‘\u001b[01m\u001b[Kwarn_unused_result\u001b[m\u001b[K’ [\u001b[01;35m\u001b[K\u001b]8;;https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html#index-Wunused-result\u0007-Wunused-result\u001b]8;;\u0007\u001b[m\u001b[K]\n",
            "18560 |         (void)\u001b[01;35m\u001b[Kfread(pstrand,sizeof(unsigned char),maxIndex,fpp)\u001b[m\u001b[K;\n",
            "      |               \u001b[01;35m\u001b[K~~~~~^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~\u001b[m\u001b[K\n",
            "\u001b[01m\u001b[KSeqTag.cpp:18561:20:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kignoring return value of ‘\u001b[01m\u001b[Ksize_t fread(void*, size_t, size_t, FILE*)\u001b[m\u001b[K’ declared with attribute ‘\u001b[01m\u001b[Kwarn_unused_result\u001b[m\u001b[K’ [\u001b[01;35m\u001b[K\u001b]8;;https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html#index-Wunused-result\u0007-Wunused-result\u001b]8;;\u0007\u001b[m\u001b[K]\n",
            "18561 |         (void)\u001b[01;35m\u001b[Kfread(nstrand,sizeof(unsigned char),maxIndex,fpn)\u001b[m\u001b[K;\n",
            "      |               \u001b[01;35m\u001b[K~~~~~^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~\u001b[m\u001b[K\n",
            "\u001b[01m\u001b[KmakeUCSCfile.cpp:\u001b[m\u001b[K In function ‘\u001b[01m\u001b[Kint main(int, char**)\u001b[m\u001b[K’:\n",
            "\u001b[01m\u001b[KmakeUCSCfile.cpp:325:37:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kignoring return value of ‘\u001b[01m\u001b[Kint system(const char*)\u001b[m\u001b[K’ declared with attribute ‘\u001b[01m\u001b[Kwarn_unused_result\u001b[m\u001b[K’ [\u001b[01;35m\u001b[K\u001b]8;;https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html#index-Wunused-result\u0007-Wunused-result\u001b]8;;\u0007\u001b[m\u001b[K]\n",
            "  325 |                         (void)\u001b[01;35m\u001b[Ksystem(str)\u001b[m\u001b[K;\n",
            "      |                               \u001b[01;35m\u001b[K~~~~~~^~~~~\u001b[m\u001b[K\n",
            "\u001b[01m\u001b[KmakeUCSCfile.cpp:336:37:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kignoring return value of ‘\u001b[01m\u001b[Kint system(const char*)\u001b[m\u001b[K’ declared with attribute ‘\u001b[01m\u001b[Kwarn_unused_result\u001b[m\u001b[K’ [\u001b[01;35m\u001b[K\u001b]8;;https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html#index-Wunused-result\u0007-Wunused-result\u001b]8;;\u0007\u001b[m\u001b[K]\n",
            "  336 |                         (void)\u001b[01;35m\u001b[Ksystem(str)\u001b[m\u001b[K;\n",
            "      |                               \u001b[01;35m\u001b[K~~~~~~^~~~~\u001b[m\u001b[K\n",
            "\u001b[01m\u001b[KmakeUCSCfile.cpp:339:37:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kignoring return value of ‘\u001b[01m\u001b[Kint system(const char*)\u001b[m\u001b[K’ declared with attribute ‘\u001b[01m\u001b[Kwarn_unused_result\u001b[m\u001b[K’ [\u001b[01;35m\u001b[K\u001b]8;;https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html#index-Wunused-result\u0007-Wunused-result\u001b]8;;\u0007\u001b[m\u001b[K]\n",
            "  339 |                         (void)\u001b[01;35m\u001b[Ksystem(str)\u001b[m\u001b[K;\n",
            "      |                               \u001b[01;35m\u001b[K~~~~~~^~~~~\u001b[m\u001b[K\n",
            "\u001b[01m\u001b[KhomerTools.cpp:\u001b[m\u001b[K In function ‘\u001b[01m\u001b[Kvoid decontaminateProgram(int, char**)\u001b[m\u001b[K’:\n",
            "\u001b[01m\u001b[KhomerTools.cpp:1703:45:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kignoring return value of ‘\u001b[01m\u001b[Kint system(const char*)\u001b[m\u001b[K’ declared with attribute ‘\u001b[01m\u001b[Kwarn_unused_result\u001b[m\u001b[K’ [\u001b[01;35m\u001b[K\u001b]8;;https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html#index-Wunused-result\u0007-Wunused-result\u001b]8;;\u0007\u001b[m\u001b[K]\n",
            " 1703 |                                 (void)\u001b[01;35m\u001b[Ksystem(\"sleep 1\")\u001b[m\u001b[K;\n",
            "      |                                       \u001b[01;35m\u001b[K~~~~~~^~~~~~~~~~~\u001b[m\u001b[K\n",
            "\u001b[01m\u001b[KhomerTools.cpp:1711:29:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kignoring return value of ‘\u001b[01m\u001b[Kint system(const char*)\u001b[m\u001b[K’ declared with attribute ‘\u001b[01m\u001b[Kwarn_unused_result\u001b[m\u001b[K’ [\u001b[01;35m\u001b[K\u001b]8;;https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html#index-Wunused-result\u0007-Wunused-result\u001b]8;;\u0007\u001b[m\u001b[K]\n",
            " 1711 |                 (void)\u001b[01;35m\u001b[Ksystem(command)\u001b[m\u001b[K;\n",
            "      |                       \u001b[01;35m\u001b[K~~~~~~^~~~~~~~~\u001b[m\u001b[K\n",
            "\u001b[01m\u001b[KhomerTools.cpp:1713:29:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kignoring return value of ‘\u001b[01m\u001b[Kint system(const char*)\u001b[m\u001b[K’ declared with attribute ‘\u001b[01m\u001b[Kwarn_unused_result\u001b[m\u001b[K’ [\u001b[01;35m\u001b[K\u001b]8;;https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html#index-Wunused-result\u0007-Wunused-result\u001b]8;;\u0007\u001b[m\u001b[K]\n",
            " 1713 |                 (void)\u001b[01;35m\u001b[Ksystem(command)\u001b[m\u001b[K;\n",
            "      |                       \u001b[01;35m\u001b[K~~~~~~^~~~~~~~~\u001b[m\u001b[K\n",
            "\u001b[01m\u001b[KhomerTools.cpp:\u001b[m\u001b[K In function ‘\u001b[01m\u001b[Kvoid specialUniqMapProgram(int, char**)\u001b[m\u001b[K’:\n",
            "\u001b[01m\u001b[KhomerTools.cpp:1788:21:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kignoring return value of ‘\u001b[01m\u001b[Kint system(const char*)\u001b[m\u001b[K’ declared with attribute ‘\u001b[01m\u001b[Kwarn_unused_result\u001b[m\u001b[K’ [\u001b[01;35m\u001b[K\u001b]8;;https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html#index-Wunused-result\u0007-Wunused-result\u001b]8;;\u0007\u001b[m\u001b[K]\n",
            " 1788 |         (void)\u001b[01;35m\u001b[Ksystem(command)\u001b[m\u001b[K;\n",
            "      |               \u001b[01;35m\u001b[K~~~~~~^~~~~~~~~\u001b[m\u001b[K\n",
            "\u001b[01m\u001b[KhomerTools.cpp:\u001b[m\u001b[K In function ‘\u001b[01m\u001b[Kvoid extractProgram(int, char**)\u001b[m\u001b[K’:\n",
            "\u001b[01m\u001b[KhomerTools.cpp:1597:17:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kargument 2 null where non-null expected [\u001b[01;35m\u001b[K\u001b]8;;https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html#index-Wnonnull\u0007-Wnonnull\u001b]8;;\u0007\u001b[m\u001b[K]\n",
            " 1597 |         if (\u001b[01;35m\u001b[Kstat(genomeDir,buf)\u001b[m\u001b[K != 0) {\n",
            "      |             \u001b[01;35m\u001b[K~~~~^~~~~~~~~~~~~~~\u001b[m\u001b[K\n",
            "In file included from \u001b[01m\u001b[KhomerTools.h:6\u001b[m\u001b[K,\n",
            "                 from \u001b[01m\u001b[KhomerTools.cpp:16\u001b[m\u001b[K:\n",
            "\u001b[01m\u001b[K/usr/include/x86_64-linux-gnu/sys/stat.h:205:12:\u001b[m\u001b[K \u001b[01;36m\u001b[Knote: \u001b[m\u001b[Kin a call to function ‘\u001b[01m\u001b[Kint stat(const char*, stat*)\u001b[m\u001b[K’ declared ‘\u001b[01m\u001b[Knonnull\u001b[m\u001b[K’\n",
            "  205 | extern int \u001b[01;36m\u001b[Kstat\u001b[m\u001b[K (const char *__restrict __file,\n",
            "      |            \u001b[01;36m\u001b[K^~~~\u001b[m\u001b[K\n",
            "\u001b[01m\u001b[KanalyzeHiC.cpp:\u001b[m\u001b[K In function ‘\u001b[01m\u001b[Kint main(int, char**)\u001b[m\u001b[K’:\n",
            "\u001b[01m\u001b[KanalyzeHiC.cpp:1375:37:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kignoring return value of ‘\u001b[01m\u001b[Kint system(const char*)\u001b[m\u001b[K’ declared with attribute ‘\u001b[01m\u001b[Kwarn_unused_result\u001b[m\u001b[K’ [\u001b[01;35m\u001b[K\u001b]8;;https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html#index-Wunused-result\u0007-Wunused-result\u001b]8;;\u0007\u001b[m\u001b[K]\n",
            " 1375 |                         (void)\u001b[01;35m\u001b[Ksystem(cmd)\u001b[m\u001b[K;\n",
            "      |                               \u001b[01;35m\u001b[K~~~~~~^~~~~\u001b[m\u001b[K\n",
            "\u001b[01m\u001b[KanalyzeHiC.cpp:1380:37:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kignoring return value of ‘\u001b[01m\u001b[Kint system(const char*)\u001b[m\u001b[K’ declared with attribute ‘\u001b[01m\u001b[Kwarn_unused_result\u001b[m\u001b[K’ [\u001b[01;35m\u001b[K\u001b]8;;https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html#index-Wunused-result\u0007-Wunused-result\u001b]8;;\u0007\u001b[m\u001b[K]\n",
            " 1380 |                         (void)\u001b[01;35m\u001b[Ksystem(cmd)\u001b[m\u001b[K;\n",
            "      |                               \u001b[01;35m\u001b[K~~~~~~^~~~~\u001b[m\u001b[K\n",
            "\u001b[01m\u001b[KanalyzeHiC.cpp:1385:37:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kignoring return value of ‘\u001b[01m\u001b[Kint system(const char*)\u001b[m\u001b[K’ declared with attribute ‘\u001b[01m\u001b[Kwarn_unused_result\u001b[m\u001b[K’ [\u001b[01;35m\u001b[K\u001b]8;;https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html#index-Wunused-result\u0007-Wunused-result\u001b]8;;\u0007\u001b[m\u001b[K]\n",
            " 1385 |                         (void)\u001b[01;35m\u001b[Ksystem(cmd)\u001b[m\u001b[K;\n",
            "      |                               \u001b[01;35m\u001b[K~~~~~~^~~~~\u001b[m\u001b[K\n",
            "\u001b[01m\u001b[KanalyzeHiC.cpp:1390:37:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kignoring return value of ‘\u001b[01m\u001b[Kint system(const char*)\u001b[m\u001b[K’ declared with attribute ‘\u001b[01m\u001b[Kwarn_unused_result\u001b[m\u001b[K’ [\u001b[01;35m\u001b[K\u001b]8;;https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html#index-Wunused-result\u0007-Wunused-result\u001b]8;;\u0007\u001b[m\u001b[K]\n",
            " 1390 |                         (void)\u001b[01;35m\u001b[Ksystem(cmd)\u001b[m\u001b[K;\n",
            "      |                               \u001b[01;35m\u001b[K~~~~~~^~~~~\u001b[m\u001b[K\n",
            "\u001b[01m\u001b[KanalyzeHiC.cpp:\u001b[m\u001b[K In function ‘\u001b[01m\u001b[Kvoid printCircosFiles(char*, int, TagLibrary*, PeakLibrary*, PeakLibrary*, int, int, int, int, char**, int, char**, int, char**, int, TagLibrary*)\u001b[m\u001b[K’:\n",
            "\u001b[01m\u001b[KanalyzeHiC.cpp:1796:29:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kignoring return value of ‘\u001b[01m\u001b[Kint system(const char*)\u001b[m\u001b[K’ declared with attribute ‘\u001b[01m\u001b[Kwarn_unused_result\u001b[m\u001b[K’ [\u001b[01;35m\u001b[K\u001b]8;;https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html#index-Wunused-result\u0007-Wunused-result\u001b]8;;\u0007\u001b[m\u001b[K]\n",
            " 1796 |                 (void)\u001b[01;35m\u001b[Ksystem(command)\u001b[m\u001b[K;\n",
            "      |                       \u001b[01;35m\u001b[K~~~~~~^~~~~~~~~\u001b[m\u001b[K\n",
            "In file included from \u001b[01m\u001b[K/usr/include/stdio.h:894\u001b[m\u001b[K,\n",
            "                 from \u001b[01m\u001b[KMotif2.h:17\u001b[m\u001b[K,\n",
            "                 from \u001b[01m\u001b[KMotif2.cpp:16\u001b[m\u001b[K:\n",
            "In function ‘\u001b[01m\u001b[Kint fprintf(FILE*, const char*, ...)\u001b[m\u001b[K’,\n",
            "    inlined from ‘\u001b[01m\u001b[Kvoid BgSequence::printAll(FILE*)\u001b[m\u001b[K’ at \u001b[01m\u001b[KMotif2.cpp:5355:9\u001b[m\u001b[K:\n",
            "\u001b[01m\u001b[K/usr/include/x86_64-linux-gnu/bits/stdio2.h:105:24:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[K‘\u001b[01m\u001b[K%s\u001b[m\u001b[K’ directive argument is null [\u001b[01;35m\u001b[K\u001b]8;;https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html#index-Wformat-overflow=\u0007-Wformat-overflow=\u001b]8;;\u0007\u001b[m\u001b[K]\n",
            "  105 |   return \u001b[01;35m\u001b[K__fprintf_chk (__stream, __USE_FORTIFY_LEVEL - 1, __fmt,\u001b[m\u001b[K\n",
            "      |          \u001b[01;35m\u001b[K~~~~~~~~~~~~~~^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~\u001b[m\u001b[K\n",
            "  106 | \u001b[01;35m\u001b[K                        __va_arg_pack ())\u001b[m\u001b[K;\n",
            "      |                         \u001b[01;35m\u001b[K~~~~~~~~~~~~~~~~~\u001b[m\u001b[K\n",
            "In function ‘\u001b[01m\u001b[Kint fprintf(FILE*, const char*, ...)\u001b[m\u001b[K’,\n",
            "    inlined from ‘\u001b[01m\u001b[Kvoid BgSequence::printFASTA(FILE*, int)\u001b[m\u001b[K’ at \u001b[01m\u001b[KMotif2.cpp:5407:9\u001b[m\u001b[K,\n",
            "    inlined from ‘\u001b[01m\u001b[Kvoid BgSequence::printFASTA(FILE*)\u001b[m\u001b[K’ at \u001b[01m\u001b[KMotif2.cpp:5396:12\u001b[m\u001b[K:\n",
            "\u001b[01m\u001b[K/usr/include/x86_64-linux-gnu/bits/stdio2.h:105:24:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[K‘\u001b[01m\u001b[K%s\u001b[m\u001b[K’ directive argument is null [\u001b[01;35m\u001b[K\u001b]8;;https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html#index-Wformat-overflow=\u0007-Wformat-overflow=\u001b]8;;\u0007\u001b[m\u001b[K]\n",
            "  105 |   return \u001b[01;35m\u001b[K__fprintf_chk (__stream, __USE_FORTIFY_LEVEL - 1, __fmt,\u001b[m\u001b[K\n",
            "      |          \u001b[01;35m\u001b[K~~~~~~~~~~~~~~^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~\u001b[m\u001b[K\n",
            "  106 | \u001b[01;35m\u001b[K                        __va_arg_pack ())\u001b[m\u001b[K;\n",
            "      |                         \u001b[01;35m\u001b[K~~~~~~~~~~~~~~~~~\u001b[m\u001b[K\n",
            "In function ‘\u001b[01m\u001b[Kint fprintf(FILE*, const char*, ...)\u001b[m\u001b[K’,\n",
            "    inlined from ‘\u001b[01m\u001b[Kvoid BgSequence::printFASTA(FILE*, int)\u001b[m\u001b[K’ at \u001b[01m\u001b[KMotif2.cpp:5407:9\u001b[m\u001b[K:\n",
            "\u001b[01m\u001b[K/usr/include/x86_64-linux-gnu/bits/stdio2.h:105:24:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[K‘\u001b[01m\u001b[K%s\u001b[m\u001b[K’ directive argument is null [\u001b[01;35m\u001b[K\u001b]8;;https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html#index-Wformat-overflow=\u0007-Wformat-overflow=\u001b]8;;\u0007\u001b[m\u001b[K]\n",
            "  105 |   return \u001b[01;35m\u001b[K__fprintf_chk (__stream, __USE_FORTIFY_LEVEL - 1, __fmt,\u001b[m\u001b[K\n",
            "      |          \u001b[01;35m\u001b[K~~~~~~~~~~~~~~^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~\u001b[m\u001b[K\n",
            "  106 | \u001b[01;35m\u001b[K                        __va_arg_pack ())\u001b[m\u001b[K;\n",
            "      |                         \u001b[01;35m\u001b[K~~~~~~~~~~~~~~~~~\u001b[m\u001b[K\n",
            "In function ‘\u001b[01m\u001b[Kint fprintf(FILE*, const char*, ...)\u001b[m\u001b[K’,\n",
            "    inlined from ‘\u001b[01m\u001b[Kvoid BgSequence::printSeq(FILE*, int)\u001b[m\u001b[K’ at \u001b[01m\u001b[KMotif2.cpp:5432:10\u001b[m\u001b[K,\n",
            "    inlined from ‘\u001b[01m\u001b[Kvoid BgSequence::printSeq(FILE*)\u001b[m\u001b[K’ at \u001b[01m\u001b[KMotif2.cpp:5423:10\u001b[m\u001b[K:\n",
            "\u001b[01m\u001b[K/usr/include/x86_64-linux-gnu/bits/stdio2.h:105:24:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[K‘\u001b[01m\u001b[K%s\u001b[m\u001b[K’ directive argument is null [\u001b[01;35m\u001b[K\u001b]8;;https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html#index-Wformat-overflow=\u0007-Wformat-overflow=\u001b]8;;\u0007\u001b[m\u001b[K]\n",
            "  105 |   return \u001b[01;35m\u001b[K__fprintf_chk (__stream, __USE_FORTIFY_LEVEL - 1, __fmt,\u001b[m\u001b[K\n",
            "      |          \u001b[01;35m\u001b[K~~~~~~~~~~~~~~^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~\u001b[m\u001b[K\n",
            "  106 | \u001b[01;35m\u001b[K                        __va_arg_pack ())\u001b[m\u001b[K;\n",
            "      |                         \u001b[01;35m\u001b[K~~~~~~~~~~~~~~~~~\u001b[m\u001b[K\n",
            "In function ‘\u001b[01m\u001b[Kint fprintf(FILE*, const char*, ...)\u001b[m\u001b[K’,\n",
            "    inlined from ‘\u001b[01m\u001b[Kvoid BgSequence::printFASTA(FILE*, int)\u001b[m\u001b[K’ at \u001b[01m\u001b[KMotif2.cpp:5407:9\u001b[m\u001b[K,\n",
            "    inlined from ‘\u001b[01m\u001b[Kvoid BgSequenceArray::printAllSequencesFASTA(FILE*)\u001b[m\u001b[K’ at \u001b[01m\u001b[KMotif2.cpp:7314:22\u001b[m\u001b[K:\n",
            "\u001b[01m\u001b[K/usr/include/x86_64-linux-gnu/bits/stdio2.h:105:24:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[K‘\u001b[01m\u001b[K%s\u001b[m\u001b[K’ directive argument is null [\u001b[01;35m\u001b[K\u001b]8;;https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html#index-Wformat-overflow=\u0007-Wformat-overflow=\u001b]8;;\u0007\u001b[m\u001b[K]\n",
            "  105 |   return \u001b[01;35m\u001b[K__fprintf_chk (__stream, __USE_FORTIFY_LEVEL - 1, __fmt,\u001b[m\u001b[K\n",
            "      |          \u001b[01;35m\u001b[K~~~~~~~~~~~~~~^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~\u001b[m\u001b[K\n",
            "  106 | \u001b[01;35m\u001b[K                        __va_arg_pack ())\u001b[m\u001b[K;\n",
            "      |                         \u001b[01;35m\u001b[K~~~~~~~~~~~~~~~~~\u001b[m\u001b[K\n",
            "In function ‘\u001b[01m\u001b[Kint fprintf(FILE*, const char*, ...)\u001b[m\u001b[K’,\n",
            "    inlined from ‘\u001b[01m\u001b[Kvoid BgSequence::printFASTA(FILE*, int)\u001b[m\u001b[K’ at \u001b[01m\u001b[KMotif2.cpp:5407:9\u001b[m\u001b[K,\n",
            "    inlined from ‘\u001b[01m\u001b[Kvoid BgSequenceArray::printSelectedSequencesFASTA(FILE*)\u001b[m\u001b[K’ at \u001b[01m\u001b[KMotif2.cpp:7334:27\u001b[m\u001b[K:\n",
            "\u001b[01m\u001b[K/usr/include/x86_64-linux-gnu/bits/stdio2.h:105:24:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[K‘\u001b[01m\u001b[K%s\u001b[m\u001b[K’ directive argument is null [\u001b[01;35m\u001b[K\u001b]8;;https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html#index-Wformat-overflow=\u0007-Wformat-overflow=\u001b]8;;\u0007\u001b[m\u001b[K]\n",
            "  105 |   return \u001b[01;35m\u001b[K__fprintf_chk (__stream, __USE_FORTIFY_LEVEL - 1, __fmt,\u001b[m\u001b[K\n",
            "      |          \u001b[01;35m\u001b[K~~~~~~~~~~~~~~^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~\u001b[m\u001b[K\n",
            "  106 | \u001b[01;35m\u001b[K                        __va_arg_pack ())\u001b[m\u001b[K;\n",
            "      |                         \u001b[01;35m\u001b[K~~~~~~~~~~~~~~~~~\u001b[m\u001b[K\n",
            "In function ‘\u001b[01m\u001b[Kint fprintf(FILE*, const char*, ...)\u001b[m\u001b[K’,\n",
            "    inlined from ‘\u001b[01m\u001b[Kvoid BgSequence::printSeq(FILE*, int)\u001b[m\u001b[K’ at \u001b[01m\u001b[KMotif2.cpp:5430:10\u001b[m\u001b[K,\n",
            "    inlined from ‘\u001b[01m\u001b[Kvoid BgSequenceArray::printSeqFile(FILE*)\u001b[m\u001b[K’ at \u001b[01m\u001b[KMotif2.cpp:7367:25\u001b[m\u001b[K:\n",
            "\u001b[01m\u001b[K/usr/include/x86_64-linux-gnu/bits/stdio2.h:105:24:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[K‘\u001b[01m\u001b[K%s\u001b[m\u001b[K’ directive argument is null [\u001b[01;35m\u001b[K\u001b]8;;https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html#index-Wformat-overflow=\u0007-Wformat-overflow=\u001b]8;;\u0007\u001b[m\u001b[K]\n",
            "  105 |   return \u001b[01;35m\u001b[K__fprintf_chk (__stream, __USE_FORTIFY_LEVEL - 1, __fmt,\u001b[m\u001b[K\n",
            "      |          \u001b[01;35m\u001b[K~~~~~~~~~~~~~~^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~\u001b[m\u001b[K\n",
            "  106 | \u001b[01;35m\u001b[K                        __va_arg_pack ())\u001b[m\u001b[K;\n",
            "      |                         \u001b[01;35m\u001b[K~~~~~~~~~~~~~~~~~\u001b[m\u001b[K\n",
            "\n",
            "\tSoftware Installed.  If not done so already, add the homer programs to your executable path.\n",
            "\n",
            "\tAdd this line to your .bash_profile or .bashrc file (or other depending on your shell):\n",
            "\t\tPATH=$PATH:/content/.//bin/\n",
            "\n",
            "\n",
            "\tSimply typing \"findMotifs.pl\" should work before running Homer.\n",
            "\n"
          ]
        }
      ],
      "source": [
        "!apt-get install ghost script\n",
        "!wget http://homer.ucsd.edu/homer/configureHomer.pl\n",
        "!perl configureHomer.pl -install"
      ]
    },
    {
      "cell_type": "code",
      "source": [
        "import os\n",
        "os.environ['PATH'] += \":/content/bin\""
      ],
      "metadata": {
        "id": "PuOt9KqUlvt5"
      },
      "execution_count": null,
      "outputs": []
    },
    {
      "cell_type": "code",
      "source": [
        "!findMotifs.pl output.fa fasta output_dir"
      ],
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "E9-GpPm6mQvF",
        "outputId": "e401ae89-980b-49c2-8f69-7f116732c619"
      },
      "execution_count": null,
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "\n",
            "Selected Options:\n",
            "\tInput file = output.fa\n",
            "\tPromoter Set = fasta\n",
            "\tOutput Directory = output_dir\n",
            "\n",
            "\t!Warning - no background FASTA file specified (Highly recommended)\n",
            "\t!Your input sequences will be randomized to serve as a background instead.\n",
            "\n",
            "\tFound 1000 sequences\n",
            "\tUsing custom gene IDs for GO analysis\n",
            "\tParsing FASTA format files...\n",
            "\tFound 1000 sequences\n",
            "\tFound 5000 sequences\n",
            "\n",
            "\tProgress: Step4 - removing redundant promoters\n",
            "\n",
            "\tProgress: Step5 - adjusting background sequences for GC/CpG content...\n",
            "\n",
            "\tSequences processed:\n",
            "\t\tAuto detected maximum sequence length of 249 bp\n",
            "\t\t6000 total\n",
            "\n",
            "\tFrequency Bins: 0.2 0.25 0.3 0.35 0.4 0.45 0.5 0.6 0.7 0.8\n",
            "\tFreq\tBin\tCount\n",
            "\t0.3\t2\t5\n",
            "\t0.35\t3\t176\n",
            "\t0.4\t4\t1084\n",
            "\t0.45\t5\t2603\n",
            "\t0.5\t6\t1688\n",
            "\t0.6\t7\t442\n",
            "\t0.7\t8\t2\n",
            "\tBin\t# Targets\t# Background\tBackground Weight\n",
            "\t3\t7\t169\t0.207\n",
            "\t4\t150\t934\t0.802\n",
            "\t5\t513\t2090\t1.226\n",
            "\t6\t298\t1390\t1.070\n",
            "\t7\t32\t410\t0.390\n",
            "\n",
            "\tNormalizing lower order oligos using homer2\n",
            "\n",
            "\tReading input files...\n",
            "\t5993 total sequences read\n",
            "\tAutonormalization: 1-mers (4 total)\n",
            "\t\tA\t28.21%\t28.21%\t1.000\n",
            "\t\tC\t21.79%\t21.79%\t1.000\n",
            "\t\tG\t21.79%\t21.79%\t1.000\n",
            "\t\tT\t28.21%\t28.21%\t1.000\n",
            "\tAutonormalization: 2-mers (16 total)\n",
            "\t\tAA\t8.95%\t8.15%\t1.098\n",
            "\t\tCA\t7.03%\t6.24%\t1.128\n",
            "\t\tGA\t6.75%\t6.03%\t1.119\n",
            "\t\tTA\t5.46%\t7.79%\t0.701\n",
            "\t\tAC\t4.69%\t6.25%\t0.751\n",
            "\t\tCC\t4.29%\t4.91%\t0.874\n",
            "\t\tGC\t6.07%\t4.60%\t1.319\n",
            "\t\tTC\t6.75%\t6.03%\t1.119\n",
            "\t\tAG\t6.08%\t6.02%\t1.011\n",
            "\t\tCG\t4.39%\t4.63%\t0.948\n",
            "\t\tGG\t4.29%\t4.91%\t0.874\n",
            "\t\tTG\t7.03%\t6.24%\t1.128\n",
            "\t\tAT\t8.50%\t7.78%\t1.092\n",
            "\t\tCT\t6.08%\t6.02%\t1.011\n",
            "\t\tGT\t4.69%\t6.25%\t0.751\n",
            "\t\tTT\t8.95%\t8.15%\t1.098\n",
            "\tAutonormalization: 3-mers (64 total)\n",
            "\tNormalization weights can be found in file: output_dir/seq.autonorm.tsv\n",
            "\tConverging on autonormalization solution:\n",
            "\t...............................................................................\n",
            "\tFinal normalization:\tAutonormalization: 1-mers (4 total)\n",
            "\t\tA\t28.21%\t28.22%\t1.000\n",
            "\t\tC\t21.79%\t21.78%\t1.000\n",
            "\t\tG\t21.79%\t21.78%\t1.000\n",
            "\t\tT\t28.21%\t28.22%\t1.000\n",
            "\tAutonormalization: 2-mers (16 total)\n",
            "\t\tAA\t8.95%\t8.45%\t1.059\n",
            "\t\tCA\t7.03%\t6.47%\t1.087\n",
            "\t\tGA\t6.75%\t6.25%\t1.078\n",
            "\t\tTA\t5.46%\t7.05%\t0.775\n",
            "\t\tAC\t4.69%\t5.74%\t0.817\n",
            "\t\tCC\t4.29%\t4.70%\t0.913\n",
            "\t\tGC\t6.07%\t5.08%\t1.195\n",
            "\t\tTC\t6.75%\t6.25%\t1.078\n",
            "\t\tAG\t6.08%\t6.02%\t1.011\n",
            "\t\tCG\t4.39%\t4.59%\t0.956\n",
            "\t\tGG\t4.29%\t4.70%\t0.913\n",
            "\t\tTG\t7.03%\t6.47%\t1.087\n",
            "\t\tAT\t8.50%\t8.01%\t1.061\n",
            "\t\tCT\t6.08%\t6.02%\t1.011\n",
            "\t\tGT\t4.69%\t5.74%\t0.817\n",
            "\t\tTT\t8.95%\t8.45%\t1.059\n",
            "\tAutonormalization: 3-mers (64 total)\n",
            "\n",
            "\tProgress: Step6 - Gene Ontology Enrichment Analysis\n",
            "\tSkipping...\n",
            "\n",
            "\tProgress: Step7 - Known motif enrichment\n",
            "\n",
            "\tReading input files...\n",
            "\t5993 total sequences read\n",
            "\t1048 motifs loaded\n",
            "\tCache length = 11180\n",
            "\tUsing hypergeometric scoring\n",
            "\tChecking enrichment of 1048 motif(s)\n",
            "\t|0%                                    50%                                  100%|\n",
            "\t=================================================================================\n",
            "\tPreparing HTML output with sequence logos...\n",
            "\t\t1 of 1048 (1e-184) PABPC1(?)/MEL-PABC1-CLIP-Seq(GSE69755)/Homer\n",
            "\t\t2 of 1048 (1e-172) polyAsite/CPSF/Homer\n",
            "\t\t3 of 1048 (1e-45) Ap4(bHLH)/AML-Tfap4-ChIP-Seq(GSE45738)/Homer\n",
            "\t\t4 of 1048 (1e-36) Mef2c(MADS)/GM12878-Mef2c-ChIP-Seq(GSE32465)/Homer\n",
            "\t\t5 of 1048 (1e-34) AT2G31460(REMB3)/col-AT2G31460-DAP-Seq(GSE60143)/Homer\n",
            "\t\t6 of 1048 (1e-33) SCL(bHLH)/HPC7-Scl-ChIP-Seq(GSE13511)/Homer\n",
            "\t\t7 of 1048 (1e-33) HuR(?)/HEK293-HuR-CLIP-Seq(GSE87887)/Homer\n",
            "\t\t8 of 1048 (1e-32) Initiator/Drosophila-Promoters/Homer\n",
            "\t\t9 of 1048 (1e-28) Hoxd12(Homeobox)/ChickenMSG-Hoxd12.Flag-ChIP-Seq(GSE86088)/Homer\n",
            "\t\t10 of 1048 (1e-26) JGL(C2H2)/col-JGL-DAP-Seq(GSE60143)/Homer\n",
            "\t\t11 of 1048 (1e-26) Foxf1(Forkhead)/Lung-Foxf1-ChIP-Seq(GSE77951)/Homer\n",
            "\t\t12 of 1048 (1e-25) Hoxa9(Homeobox)/ChickenMSG-Hoxa9.Flag-ChIP-Seq(GSE86088)/Homer\n",
            "\t\t13 of 1048 (1e-25) Tcf12(bHLH)/GM12878-Tcf12-ChIP-Seq(GSE32465)/Homer\n",
            "\t\t14 of 1048 (1e-24) STZ(C2H2)/colamp-STZ-DAP-Seq(GSE60143)/Homer\n",
            "\t\t15 of 1048 (1e-23) PHA-4(Forkhead)/cElegans-Embryos-PHA4-ChIP-Seq(modEncode)/Homer\n",
            "\t\t16 of 1048 (1e-22) SeqBias: TA-repeat\n",
            "\t\t17 of 1048 (1e-22) MyoG(bHLH)/C2C12-MyoG-ChIP-Seq(GSE36024)/Homer\n",
            "\t\t18 of 1048 (1e-21) Tcf21(bHLH)/ArterySmoothMuscle-Tcf21-ChIP-Seq(GSE61369)/Homer\n",
            "\t\t19 of 1048 (1e-20) Mef2b(MADS)/HEK293-Mef2b.V5-ChIP-Seq(GSE67450)/Homer\n",
            "\t\t20 of 1048 (1e-19) At5g04390(C2H2)/col200-At5g04390-DAP-Seq(GSE60143)/Homer\n",
            "\t\t21 of 1048 (1e-19) bZIP18(bZIP)/colamp-bZIP18-DAP-Seq(GSE60143)/Homer\n",
            "\t\t22 of 1048 (1e-18) FoxL2(Forkhead)/Ovary-FoxL2-ChIP-Seq(GSE60858)/Homer\n",
            "\t\t23 of 1048 (1e-18) bZIP52(bZIP)/colamp-bZIP52-DAP-Seq(GSE60143)/Homer\n",
            "\t\t24 of 1048 (1e-18) SEP3(MADS)/Arabidoposis-Flower-Sep3-ChIP-Seq/Homer\n",
            "\t\t25 of 1048 (1e-17) hINR(CPE)\n",
            "\t\t26 of 1048 (1e-16) FOXA1(Forkhead)/MCF7-FOXA1-ChIP-Seq(GSE26831)/Homer\n",
            "\t\t27 of 1048 (1e-15) Hoxd10(Homeobox)/ChickenMSG-Hoxd10.Flag-ChIP-Seq(GSE86088)/Homer\n",
            "\t\t28 of 1048 (1e-15) FOXA1(Forkhead)/LNCAP-FOXA1-ChIP-Seq(GSE27824)/Homer\n",
            "\t\t29 of 1048 (1e-15) RLR1?/SacCer-Promoters/Homer\n",
            "\t\t30 of 1048 (1e-15) FOXM1(Forkhead)/MCF7-FOXM1-ChIP-Seq(GSE72977)/Homer\n",
            "\t\t31 of 1048 (1e-15) Mef2d(MADS)/Retina-Mef2d-ChIP-Seq(GSE61391)/Homer\n",
            "\t\t32 of 1048 (1e-14) Bapx1(Homeobox)/VertebralCol-Bapx1-ChIP-Seq(GSE36672)/Homer\n",
            "\t\t33 of 1048 (1e-14) ZBTB38(Zf)/Hela-ZBTB38-ChIP-seq(GSE108618)/Homer\n",
            "\t\t34 of 1048 (1e-14) Nkx2.2(Homeobox)/NPC-Nkx2.2-ChIP-Seq(GSE61673)/Homer\n",
            "\t\t35 of 1048 (1e-14) RIN(MADS)/Tomato-RIN-ChIP-Seq(GSE116581)/Homer\n",
            "\t\t36 of 1048 (1e-14) E2A(bHLH)/proBcell-E2A-ChIP-Seq(GSE21978)/Homer\n",
            "\t\t37 of 1048 (1e-12) Ptf1a(bHLH)/Panc1-Ptf1a-ChIP-Seq(GSE47459)/Homer\n",
            "\t\t38 of 1048 (1e-12) MyoD(bHLH)/Myotube-MyoD-ChIP-Seq(GSE21614)/Homer\n",
            "\t\t39 of 1048 (1e-11) Myf5(bHLH)/GM-Myf5-ChIP-Seq(GSE24852)/Homer\n",
            "\t\t40 of 1048 (1e-11) FOXK1(Forkhead)/HEK293-FOXK1-ChIP-Seq(GSE51673)/Homer\n",
            "\t\t41 of 1048 (1e-11) BHLHA15(bHLH)/NIH3T3-BHLHB8.HA-ChIP-Seq(GSE119782)/Homer\n",
            "\t\t42 of 1048 (1e-11) Ascl2(bHLH)/ESC-Ascl2-ChIP-Seq(GSE97712)/Homer\n",
            "\t\t43 of 1048 (1e-11) Cdx2(Homeobox)/mES-Cdx2-ChIP-Seq(GSE14586)/Homer\n",
            "\t\t44 of 1048 (1e-10) Twist2(bHLH)/Myoblast-Twist2.Ty1-ChIP-Seq(GSE127998)/Homer\n",
            "\t\t45 of 1048 (1e-10) Hoxc13(Homeobox)/EB-Hoxc13.HA-ChIP-Seq(GSE142377)/Homer\n",
            "\t\t46 of 1048 (1e-10) SOL1(CPP)/colamp-SOL1-DAP-Seq(GSE60143)/Homer\n",
            "\t\t47 of 1048 (1e-9) Mef2a(MADS)/HL1-Mef2a.biotin-ChIP-Seq(GSE21529)/Homer\n",
            "\t\t48 of 1048 (1e-9) MafF(bZIP)/HepG2-MafF-ChIP-Seq(GSE31477)/Homer\n",
            "\t\t49 of 1048 (1e-9) MafA(bZIP)/Islet-MafA-ChIP-Seq(GSE30298)/Homer\n",
            "\t\t50 of 1048 (1e-9) GATA20(C2C2gata)/colamp-GATA20-DAP-Seq(GSE60143)/Homer\n",
            "\t\t51 of 1048 (1e-9) Hoxa13(Homeobox)/ChickenMSG-Hoxa13.Flag-ChIP-Seq(GSE86088)/Homer\n",
            "\t\t52 of 1048 (1e-9) HEB(bHLH)/mES-Heb-ChIP-Seq(GSE53233)/Homer\n",
            "\t\t53 of 1048 (1e-9) Foxo3(Forkhead)/U2OS-Foxo3-ChIP-Seq(E-MTAB-2701)/Homer\n",
            "\t\t54 of 1048 (1e-9) Foxh1(Forkhead)/hESC-FOXH1-ChIP-Seq(GSE29422)/Homer\n",
            "\t\t55 of 1048 (1e-9) ERF15(AP2EREBP)/colamp-ERF15-DAP-Seq(GSE60143)/Homer\n",
            "\t\t56 of 1048 (1e-9) Hoxc6(Homeobox)/EB-Hoxc6.iFlag-ChIP-Seq(GSE142377)/Homer\n",
            "\t\t57 of 1048 (1e-9) ANL2(HB)/col-ANL2-DAP-Seq(GSE60143)/Homer\n",
            "\t\t58 of 1048 (1e-9) Unknown2/Drosophila-Promoters/Homer\n",
            "\t\t59 of 1048 (1e-8) Nkx2.5(Homeobox)/HL1-Nkx2.5.biotin-ChIP-Seq(GSE21529)/Homer\n",
            "\t\t60 of 1048 (1e-8) AT2G20110(CPP)/colamp-AT2G20110-DAP-Seq(GSE60143)/Homer\n",
            "\t\t61 of 1048 (1e-8) Ascl1(bHLH)/NeuralTubes-Ascl1-ChIP-Seq(GSE55840)/Homer\n",
            "\t\t62 of 1048 (1e-8) Oct6(POU,Homeobox)/NPC-Pou3f1-ChIP-Seq(GSE35496)/Homer\n",
            "\t\t63 of 1048 (1e-8) HOXD13(Homeobox)/Chicken-Hoxd13-ChIP-Seq(GSE38910)/Homer\n",
            "\t\t64 of 1048 (1e-8) At5g47390(MYBrelated)/col-At5g47390-DAP-Seq(GSE60143)/Homer\n",
            "\t\t65 of 1048 (1e-8) Hoxa10(Homeobox)/ChickenMSG-Hoxa10.Flag-ChIP-Seq(GSE86088)/Homer\n",
            "\t\t66 of 1048 (1e-8) KANADI1(Myb)/Seedling-KAN1-ChIP-Seq(GSE48081)/Homer\n",
            "\t\t67 of 1048 (1e-8) TCX2(CPP)/colamp-TCX2-DAP-Seq(GSE60143)/Homer\n",
            "\t\t68 of 1048 (1e-8) AT5G61620(MYBrelated)/colamp-AT5G61620-DAP-Seq(GSE60143)/Homer\n",
            "\t\t69 of 1048 (1e-7) ZBTB18(Zf)/HEK293-ZBTB18.GFP-ChIP-Seq(GSE58341)/Homer\n",
            "\t\t70 of 1048 (1e-7) AZF1(C2H2)/colamp-AZF1-DAP-Seq(GSE60143)/Homer\n",
            "\t\t71 of 1048 (1e-7) HOXB13(Homeobox)/ProstateTumor-HOXB13-ChIP-Seq(GSE56288)/Homer\n",
            "\t\t72 of 1048 (1e-7) At1g74840(MYBrelated)/col100-At1g74840-DAP-Seq(GSE60143)/Homer\n",
            "\t\t73 of 1048 (1e-7) TCF4(bHLH)/SHSY5Y-TCF4-ChIP-Seq(GSE96915)/Homer\n",
            "\t\t74 of 1048 (1e-6) PU.1-IRF(ETS:IRF)/Bcell-PU.1-ChIP-Seq(GSE21512)/Homer\n",
            "\t\t75 of 1048 (1e-6) Atoh7(bHLH)/Retina-Atoh7-CutnRun(GSE156756)/Homer\n",
            "\t\t76 of 1048 (1e-6) TAGL1(MADS)/Tomato-TAGL1-ChIP-Seq(GSE116581)/Homer\n",
            "\t\t77 of 1048 (1e-6) Bach2(bZIP)/OCILy7-Bach2-ChIP-Seq(GSE44420)/Homer\n",
            "\t\t78 of 1048 (1e-6) Pit1(Homeobox)/GCrat-Pit1-ChIP-Seq(GSE58009)/Homer\n",
            "\t\t79 of 1048 (1e-6) Oct2(POU,Homeobox)/Bcell-Oct2-ChIP-Seq(GSE21512)/Homer\n",
            "\t\t80 of 1048 (1e-6) FoxD3(forkhead)/ZebrafishEmbryo-Foxd3.biotin-ChIP-seq(GSE106676)/Homer\n",
            "\t\t81 of 1048 (1e-6) GATA19(C2C2gata)/colamp-GATA19-DAP-Seq(GSE60143)/Homer\n",
            "\t\t82 of 1048 (1e-6) AT3G57600(AP2EREBP)/col-AT3G57600-DAP-Seq(GSE60143)/Homer\n",
            "\t\t83 of 1048 (1e-6) Mesp1(bHLH)/ESC-Mesp1-ChIP-Seq(GSE165102)/Homer\n",
            "\t\t84 of 1048 (1e-6) caudal(Homeobox)/Drosophila-Embryos-ChIP-Chip(modEncode)/Homer\n",
            "\t\t85 of 1048 (1e-6) ZNF416(Zf)/HEK293-ZNF416.GFP-ChIP-Seq(GSE58341)/Homer\n",
            "\t\t86 of 1048 (1e-5) SeqBias: CG bias\n",
            "\t\t87 of 1048 (1e-5) LIN-39(Homeobox)/cElegans.L3-LIN39-ChIP-Seq(modEncode)/Homer\n",
            "\t\t88 of 1048 (1e-5) HDG1(Homeobox)/col100-HDG1-DAP-Seq(GSE60143)/Homer\n",
            "\t\t89 of 1048 (1e-5) NFATC2(RHD)/Islets-NFATC2-ChIP-Seq(GSE158496)/Homer\n",
            "\t\t90 of 1048 (1e-5) HNF6(Homeobox)/Liver-Hnf6-ChIP-Seq(ERP000394)/Homer\n",
            "\t\t91 of 1048 (1e-5) ASL18(LOBAS2)/colamp-ASL18-DAP-Seq(GSE60143)/Homer\n",
            "\t\t92 of 1048 (1e-5) bZIP:IRF(bZIP,IRF)/Th17-BatF-ChIP-Seq(GSE39756)/Homer\n",
            "\t\t93 of 1048 (1e-5) VIP1(bZIP)/col-VIP1-DAP-Seq(GSE60143)/Homer\n",
            "\t\t94 of 1048 (1e-5) Smad3(MAD)/NPC-Smad3-ChIP-Seq(GSE36673)/Homer\n",
            "\t\t95 of 1048 (1e-5) EGL-5(Homeobox)/cElegans-L3-EGL5-ChIP-Seq(modEncode)/Homer\n",
            "\t\t96 of 1048 (1e-5) AGL63(MADS)/col-AGL63-DAP-Seq(GSE60143)/Homer\n",
            "\t\t97 of 1048 (1e-5) Cux2(Homeobox)/Liver-Cux2-ChIP-Seq(GSE35985)/Homer\n",
            "\t\t98 of 1048 (1e-5) Hoxa11(Homeobox)/ChickenMSG-Hoxa11.Flag-ChIP-Seq(GSE86088)/Homer\n",
            "\t\t99 of 1048 (1e-5) At1g19000(MYBrelated)/colamp-At1g19000-DAP-Seq(GSE60143)/Homer\n",
            "\t\t100 of 1048 (1e-4) KAN2(G2like)/colamp-KAN2-DAP-Seq(GSE60143)/Homer\n",
            "\t\t101 of 1048 (1e-4) GATA4(C2C2gata)/col-GATA4-DAP-Seq(GSE60143)/Homer\n",
            "\t\t102 of 1048 (1e-4) Oct11(POU,Homeobox)/NCIH1048-POU2F3-ChIP-seq(GSE115123)/Homer\n",
            "\t\t103 of 1048 (1e-4) AP-1(bZIP)/ThioMac-PU.1-ChIP-Seq(GSE21512)/Homer\n",
            "\t\t104 of 1048 (1e-4) Nkx2.1(Homeobox)/LungAC-Nkx2.1-ChIP-Seq(GSE43252)/Homer\n",
            "\t\t105 of 1048 (1e-4) IRF3(IRF)/BMDM-Irf3-ChIP-Seq(GSE67343)/Homer\n",
            "\t\t106 of 1048 (1e-4) ZNF322(Zf)/HEK293-ZNF322.GFP-ChIP-Seq(GSE58341)/Homer\n",
            "\t\t107 of 1048 (1e-4) HIC1(Zf)/Treg-ZBTB29-ChIP-Seq(GSE99889)/Homer\n",
            "\t\t108 of 1048 (1e-4) Hoxc10(Homeobox)/EB-Hoxc10.iFlag-ChIP-Seq(GSE142377)/Homer\n",
            "\t\t109 of 1048 (1e-3) bZIP69(bZIP)/col-bZIP69-DAP-Seq(GSE60143)/Homer\n",
            "\t\t110 of 1048 (1e-3) bHLH80(bHLH)/col-bHLH80-DAP-Seq(GSE60143)/Homer\n",
            "\t\t111 of 1048 (1e-3) Fra2(bZIP)/Striatum-Fra2-ChIP-Seq(GSE43429)/Homer\n",
            "\t\t112 of 1048 (1e-3) ZNF7(Zf)/HepG2-ZNF7.Flag-ChIP-Seq(Encode)/Homer\n",
            "\t\t113 of 1048 (1e-3) Atoh1(bHLH)/Cerebellum-Atoh1-ChIP-Seq(GSE22111)/Homer\n",
            "\t\t114 of 1048 (1e-3) WUS1(Homeobox)/colamp-WUS1-DAP-Seq(GSE60143)/Homer\n",
            "\t\t115 of 1048 (1e-3) HDG7(HB)/col-HDG7-DAP-Seq(GSE60143)/Homer\n",
            "\t\t116 of 1048 (1e-3) FOXK2(Forkhead)/U2OS-FOXK2-ChIP-Seq(E-MTAB-2204)/Homer\n",
            "\t\t117 of 1048 (1e-3) AT4G00250(GeBP)/col-AT4G00250-DAP-Seq(GSE60143)/Homer\n",
            "\t\t118 of 1048 (1e-3) Sox7(HMG)/ESC-Sox7-ChIP-Seq(GSE133899)/Homer\n",
            "\t\t119 of 1048 (1e-3) Hoxc9(Homeobox)/Ainv15-Hoxc9-ChIP-Seq(GSE21812)/Homer\n",
            "\t\t120 of 1048 (1e-3) Sox17(HMG)/Endoderm-Sox17-ChIP-Seq(GSE61475)/Homer\n",
            "\t\t121 of 1048 (1e-3) Lola-I(Zf)/Embryo-LolaI-ChIP-Seq(GSE200870)/Homer\n",
            "\t\t122 of 1048 (1e-3) AMYB(HTH)/Testes-AMYB-ChIP-Seq(GSE44588)/Homer\n",
            "\t\t123 of 1048 (1e-3) Oct4(POU,Homeobox)/mES-Oct4-ChIP-Seq(GSE11431)/Homer\n",
            "\t\t124 of 1048 (1e-3) NFAT(RHD)/Jurkat-NFATC1-ChIP-Seq(Jolma_et_al.)/Homer\n",
            "\t\t125 of 1048 (1e-3) At5g08520(MYBrelated)/colamp-At5g08520-DAP-Seq(GSE60143)/Homer\n",
            "\t\t126 of 1048 (1e-3) EIN3(EIL)/col-EIN3-DAP-Seq(GSE60143)/Homer\n",
            "\t\t127 of 1048 (1e-3) LBD23(LOBAS2)/colamp-LBD23-DAP-Seq(GSE60143)/Homer\n",
            "\t\t128 of 1048 (1e-3) ZNF415(Zf)/HEK293-ZNF415.GFP-ChIP-Seq(GSE58341)/Homer\n",
            "\t\t129 of 1048 (1e-3) Egr1(Zf)/K562-Egr1-ChIP-Seq(GSE32465)/Homer\n",
            "\t\t130 of 1048 (1e-3) GRF9(GRF)/colamp-GRF9-DAP-Seq(GSE60143)/Homer\n",
            "\t\t131 of 1048 (1e-2) At5g58900(MYBrelated)/colamp-At5g58900-DAP-Seq(GSE60143)/Homer\n",
            "\t\t132 of 1048 (1e-2) Tal1\n",
            "\t\t133 of 1048 (1e-2) PU.1:IRF8(ETS:IRF)/pDC-Irf8-ChIP-Seq(GSE66899)/Homer\n",
            "\t\t134 of 1048 (1e-2) bHLH122(bHLH)/col100-bHLH122-DAP-Seq(GSE60143)/Homer\n",
            "\t\t135 of 1048 (1e-2) AT5G56840(MYBrelated)/colamp-AT5G56840-DAP-Seq(GSE60143)/Homer\n",
            "\t\t136 of 1048 (1e-2) Atf3(bZIP)/GBM-ATF3-ChIP-Seq(GSE33912)/Homer\n",
            "\t\t137 of 1048 (1e-2) At1g49010(MYBrelated)/col-At1g49010-DAP-Seq(GSE60143)/Homer\n",
            "\t\t138 of 1048 (1e-2) MYB73(MYB)/col-MYB73-DAP-Seq(GSE60143)/Homer\n",
            "\t\t139 of 1048 (1e-2) BATF(bZIP)/Th17-BATF-ChIP-Seq(GSE39756)/Homer\n",
            "\t\t140 of 1048 (1e-2) CHR(?)/Hela-CellCycle-Expression/Homer\n",
            "\t\t141 of 1048 (1e-2) AT2G15740(C2H2)/col-AT2G15740-DAP-Seq(GSE60143)/Homer\n",
            "\t\t142 of 1048 (1e-2) Hoxd9(Homeobox)/EB-Hoxd9.HA-ChIP-Seq(GSE142377)/Homer\n",
            "\t\t143 of 1048 (1e-2) Fosl2(bZIP)/3T3L1-Fosl2-ChIP-Seq(GSE56872)/Homer\n",
            "\t\t144 of 1048 (1e-2) Zic3(Zf)/mES-Zic3-ChIP-Seq(GSE37889)/Homer\n",
            "\t\t145 of 1048 (1e-2) PCF/Arabidopsis-Promoters/Homer\n",
            "\t\t146 of 1048 (1e-2) Jun-AP1(bZIP)/K562-cJun-ChIP-Seq(GSE31477)/Homer\n",
            "\t\t147 of 1048 (1e-2) JunB(bZIP)/DendriticCells-Junb-ChIP-Seq(GSE36099)/Homer\n",
            "\t\t148 of 1048 (1e-2) TOD6?/SacCer-Promoters/Homer\n",
            "\t\t149 of 1048 (1e-2) WRKY25(WRKY)/colamp-WRKY25-DAP-Seq(GSE60143)/Homer\n",
            "\t\t150 of 1048 (1e-2) Unknown6/Drosophila-Promoters/Homer\n",
            "\t\t151 of 1048 (1e-2) NeuroG2(bHLH)/Fibroblast-NeuroG2-ChIP-Seq(GSE75910)/Homer\n",
            "\t\t152 of 1048 (1e-2) MYB88(MYB)/col-MYB88-DAP-Seq(GSE60143)/Homer\n",
            "\t\t153 of 1048 (1e-2) NFE2L2(bZIP)/HepG2-NFE2L2-ChIP-Seq(Encode)/Homer\n",
            "\t\t154 of 1048 (1e-2) AT5G23930(mTERF)/col-AT5G23930-DAP-Seq(GSE60143)/Homer\n",
            "\t\t155 of 1048 (1e-2) YY1(Zf)/Promoter/Homer\n",
            "\t\t156 of 1048 (1e-2) Sox2(HMG)/mES-Sox2-ChIP-Seq(GSE11431)/Homer\n",
            "\t\t157 of 1048 (1e-2) Bcl11a(Zf)/HSPC-BCL11A-ChIP-Seq(GSE104676)/Homer\n",
            "\t\t158 of 1048 (1e-2) Meis1(Homeobox)/MastCells-Meis1-ChIP-Seq(GSE48085)/Homer\n",
            "\t\t159 of 1048 (1e-2) CUX1(Homeobox)/K562-CUX1-ChIP-Seq(GSE92882)/Homer\n",
            "\t\t160 of 1048 (1e-2) ASHR1(ND)/col-ASHR1-DAP-Seq(GSE60143)/Homer\n",
            "\t\t161 of 1048 (1e-2) IRF8(IRF)/BMDM-IRF8-ChIP-Seq(GSE77884)/Homer\n",
            "\t\t162 of 1048 (1e-2) WRKY18(WRKY)/col-WRKY18-DAP-Seq(GSE60143)/Homer\n",
            "\n",
            "\tProgress: Step8 - De novo motif finding (HOMER)\n",
            "\n",
            "\tScanning input files...\n",
            "\tParsing sequences...\n",
            "\t|0%                                   50%                                  100%|\n",
            "\t================================================================================\n",
            "\tTotal number of Oligos: 32896\n",
            "\tAutoadjustment for sequence coverage in background: 1.00x\n",
            "\n",
            "\tOligos: 32896 of 34497 max\n",
            "\tTree  : 63700 of 172485 max\n",
            "\tOptimizing memory usage...\n",
            "\tCache length = 11180\n",
            "\tUsing hypergeometric scoring\n",
            "\n",
            "\tGlobal Optimization Phase: Looking for enriched oligos with up to 1 mismatches...\n",
            "\n",
            "\tScreening oligos 32896 (allowing 0 mismatches):\n",
            "\t|0%                                   50%                                  100%|\n",
            "\t================================================================================\n",
            "\t\t6.45% skipped, 93.55% checked (30773 of 32896), of those checked:\n",
            "\t\t6.45% not in target, 0.00% increased p-value, 0.00% high p-value\n",
            "\n",
            "\tScreening oligos 32896 (allowing 1 mismatches):\n",
            "\t|0%                                   50%                                  100%|\n",
            "\t================================================================================\n",
            "\t\t6.45% skipped, 93.55% checked (30773 of 32896), of those checked:\n",
            "\t\t0.00% not in target, 47.41% increased p-value, 58.41% high p-value\n",
            "\tReading input files...\n",
            "\t5993 total sequences read\n",
            "\tCache length = 11180\n",
            "\tUsing hypergeometric scoring\n",
            "\n",
            "\tLocal Optimization Phase:\n",
            "\t1 of 25 Initial Sequence: AAAATGAA... (-402.360)\n",
            "\t\tRound 1: -733.35 AAAATGAA T:1892.0(84.94%),B:1009.9(20.94%),P:1e-318\n",
            "\t\tRound 2: -733.35 AAAATGAA T:1892.0(84.94%),B:1009.9(20.94%),P:1e-318\n",
            "\t\t=Final=: -668.55 AAAATGAA T:861.0(86.10%),B:1059.4(24.67%),P:1e-290\n",
            "\t\tPerforming exhaustive masking of motif...\n",
            "\t\tReprioritizing potential motifs...\n",
            "\t2 of 25 Initial Sequence: AGCGCTGG... (-221.181)\n",
            "\t\tRound 1: -323.08 AGCGCTGG T:748.0(52.69%),B:627.0(13.59%),P:1e-140\n",
            "\t\tRound 2: -323.08 AGCGCTGG T:748.0(52.69%),B:627.0(13.59%),P:1e-140\n",
            "\t\t=Final=: -234.14 AGCGCTGG T:483.0(48.30%),B:650.9(15.16%),P:1e-101\n",
            "\t\tPerforming exhaustive masking of motif...\n",
            "\t\tReprioritizing potential motifs...\n",
            "\t3 of 25 Initial Sequence: CAGCTGCA... (-199.177)\n",
            "\t\tRound 1: -352.01 CAGCTGCA T:1176.0(69.17%),B:1197.8(24.33%),P:1e-152\n",
            "\t\tRound 2: -368.83 MARCTGSM T:1571.0(79.23%),B:1707.3(32.80%),P:1e-160\n",
            "\t\tRound 3: -388.37 CAGCTGGA T:1768.0(82.95%),B:1887.3(35.56%),P:1e-168\n",
            "\t\tRound 4: -388.37 CAGCTGGA T:1768.0(82.95%),B:1887.3(35.56%),P:1e-168\n",
            "\t\t=Final=: -368.30 CAGCTGGA T:808.0(80.80%),B:1482.6(34.53%),P:1e-159\n",
            "\t\tPerforming exhaustive masking of motif...\n",
            "\t\tReprioritizing potential motifs...\n",
            "\t4 of 25 Initial Sequence: AAAATGGG... (-125.710)\n",
            "\t\tRound 1: -269.83 AAAATGGG T:910.0(59.77%),B:1034.2(21.40%),P:1e-117\n",
            "\t\tRound 2: -273.33 CAAATGSC T:751.0(52.83%),B:755.9(16.12%),P:1e-118\n",
            "\t\tRound 3: -273.33 CAAATGSC T:751.0(52.83%),B:755.9(16.12%),P:1e-118\n",
            "\t\t=Final=: -263.78 CAAATGSC T:518.0(51.80%),B:684.1(15.93%),P:1e-114\n",
            "\t\tPerforming exhaustive masking of motif...\n",
            "\t\tReprioritizing potential motifs...\n",
            "\t5 of 25 Initial Sequence: TCGGCAGT... (-72.618)\n",
            "\t\tRound 1: -159.86 TCGGCAGT T:707.0(50.71%),B:1057.0(21.80%),P:1e-69\n",
            "\t\tRound 2: -184.64 TSCSCATY T:1181.0(69.32%),B:1931.9(36.22%),P:1e-80\n",
            "\t\tRound 3: -219.65 TSGGCAKY T:1071.0(65.75%),B:1526.4(29.91%),P:1e-95\n",
            "\t\tRound 4: -219.65 TSGGCAKY T:1071.0(65.75%),B:1526.4(29.91%),P:1e-95\n",
            "\t\t=Final=: -181.49 TSGGCAKY T:638.0(63.80%),B:1341.9(31.25%),P:1e-78\n",
            "\t\tPerforming exhaustive masking of motif...\n",
            "\t\tReprioritizing potential motifs...\n",
            "\t6 of 25 Initial Sequence: GACTGGAG... (-63.608)\n",
            "\t\tRound 1: -160.06 GACTGGAG T:929.0(60.52%),B:1538.5(30.11%),P:1e-69\n",
            "\t\tRound 2: -164.61 GAMTGRAG T:1182.0(69.35%),B:2057.4(38.06%),P:1e-71\n",
            "\t\tRound 3: -164.61 GAMTGRAG T:1182.0(69.35%),B:2057.4(38.06%),P:1e-71\n",
            "\t\t=Final=: -150.15 GAMTGRAG T:675.0(67.50%),B:1617.7(37.67%),P:1e-65\n",
            "\t\tPerforming exhaustive masking of motif...\n",
            "\t\tReprioritizing potential motifs...\n",
            "\t7 of 25 Initial Sequence: AATAAATG... (-42.012)\n",
            "\t\tRound 1: -98.92 AATAAATG T:288.0(25.03%),B:366.4(8.17%),P:1e-42\n",
            "\t\tRound 2: -98.92 AATAAATG T:288.0(25.03%),B:366.4(8.17%),P:1e-42\n",
            "\t\t=Final=: -102.42 AATAAATG T:254.0(25.40%),B:351.2(8.18%),P:1e-44\n",
            "\t\tPerforming exhaustive masking of motif...\n",
            "\t\tReprioritizing potential motifs...\n",
            "\t8 of 25 Initial Sequence: GGCTCAGC... (-32.584)\n",
            "\t\tRound 1: -72.12 GGCTCAGC T:509.0(39.91%),B:1032.4(21.36%),P:1e-31\n",
            "\t\tRound 2: -81.38 GGCTCAGC T:679.0(49.30%),B:1426.9(28.26%),P:1e-35\n",
            "\t\tRound 3: -90.53 SGCTSAGC T:844.0(57.02%),B:1792.7(34.12%),P:1e-39\n",
            "\t\tRound 4: -90.53 SGCTSAGC T:844.0(57.02%),B:1792.7(34.12%),P:1e-39\n",
            "\t\t=Final=: -62.36 SGCTSAGC T:302.0(30.20%),B:635.4(14.80%),P:1e-27\n",
            "\t\tPerforming exhaustive masking of motif...\n",
            "\t\tReprioritizing potential motifs...\n",
            "\t9 of 25 Initial Sequence: GATAAAAT... (-21.697)\n",
            "\t\tRound 1: -59.70 RATAAACT T:371.0(31.01%),B:734.6(15.71%),P:1e-25\n",
            "\t\tRound 2: -59.88 GATGAAAT T:500.0(39.36%),B:1088.5(22.38%),P:1e-26\n",
            "\t\tRound 3: -66.59 GATCAAAT T:681.0(49.41%),B:1549.3(30.28%),P:1e-28\n",
            "\t\tRound 4: -66.59 GATCAAAT T:681.0(49.41%),B:1549.3(30.28%),P:1e-28\n",
            "\t\t=Final=: -61.24 GATCAAAT T:469.0(46.90%),B:1237.0(28.81%),P:1e-26\n",
            "\t\tPerforming exhaustive masking of motif...\n",
            "\t\tReprioritizing potential motifs...\n",
            "\t10 of 25 Initial Sequence: AGCAATAA... (-19.078)\n",
            "\t\tRound 1: -31.35 AGCAATAA T:346.0(29.26%),B:860.5(18.15%),P:1e-13\n",
            "\t\tRound 2: -49.58 HMRAATRA T:366.0(30.66%),B:780.9(16.61%),P:1e-21\n",
            "\t\tRound 3: -52.31 YMRAATRA T:424.0(34.57%),B:927.6(19.42%),P:1e-22\n",
            "\t\tRound 4: -62.10 HMRAATGA T:533.0(41.33%),B:1167.3(23.80%),P:1e-26\n",
            "\t\tRound 5: -62.10 HMRAATGA T:533.0(41.33%),B:1167.3(23.80%),P:1e-26\n",
            "\t\t=Final=: -66.24 HMRAATGA T:430.0(43.00%),B:1060.4(24.69%),P:1e-28\n",
            "\t\tPerforming exhaustive masking of motif...\n",
            "\t\tReprioritizing potential motifs...\n",
            "\t11 of 25 Initial Sequence: AGCTCGAG... (-18.079)\n",
            "\t\tRound 1: -41.68 AGCTCGAG T:548.0(42.21%),B:1390.8(27.65%),P:1e-18\n",
            "\t\tRound 2: -41.68 AGCTCGAG T:548.0(42.21%),B:1390.8(27.65%),P:1e-18\n",
            "\t\t=Final=: -36.64 AGCTCGAG T:413.0(41.30%),B:1191.3(27.74%),P:1e-15\n",
            "\t\tPerforming exhaustive masking of motif...\n",
            "\t\tReprioritizing potential motifs...\n",
            "\t12 of 25 Initial Sequence: CAGACTGC... (-15.424)\n",
            "\t\tRound 1: -51.61 CAGACTGC T:400.0(32.98%),B:862.6(18.19%),P:1e-22\n",
            "\t\tRound 2: -69.12 CAGACTGS T:480.0(38.14%),B:971.1(20.24%),P:1e-30\n",
            "\t\tRound 3: -69.12 CAGACTGS T:480.0(38.14%),B:971.1(20.24%),P:1e-30\n",
            "\t\t=Final=: -52.71 CAGACTGS T:377.0(37.70%),B:946.0(22.03%),P:1e-22\n",
            "\t\tPerforming exhaustive masking of motif...\n",
            "\t\tReprioritizing potential motifs...\n",
            "\t13 of 25 Initial Sequence: TAGCGATC... (-15.004)\n",
            "\t\tRound 1: -23.42 TAGCGATC T:69.0(6.67%),B:99.8(2.28%),P:1e-10\n",
            "\t\tRound 2: -23.77 TARCAATA T:184.0(16.81%),B:422.0(9.34%),P:1e-10\n",
            "\t\tRound 3: -26.07 TARCAATA T:162.0(14.96%),B:340.3(7.61%),P:1e-11\n",
            "\t\tRound 4: -26.07 TARCAATA T:162.0(14.96%),B:340.3(7.61%),P:1e-11\n",
            "\t\t=Final=: -26.90 TARCAATA T:152.0(15.20%),B:331.1(7.71%),P:1e-11\n",
            "\t\tPerforming exhaustive masking of motif...\n",
            "\t\tReprioritizing potential motifs...\n",
            "\t14 of 25 Initial Sequence: CACTCGCA... (-14.659)\n",
            "\t\tRound 1: -35.70 CACTCGCD T:294.0(25.48%),B:666.7(14.37%),P:1e-15\n",
            "\t\tRound 2: -42.58 VACTCSVN T:793.0(54.77%),B:2133.1(39.15%),P:1e-18\n",
            "\t\tRound 3: -42.58 VACTCSVN T:793.0(54.77%),B:2133.1(39.15%),P:1e-18\n",
            "\t\t=Final=: -41.42 VACTCSVN T:560.0(56.00%),B:1745.7(40.65%),P:1e-17\n",
            "\t\tPerforming exhaustive masking of motif...\n",
            "\t\tReprioritizing potential motifs...\n",
            "\t15 of 25 Initial Sequence: CCCTATTT... (-13.031)\n",
            "\t\tRound 1: -35.49 CCCTATTT T:108.0(10.24%),B:154.5(3.52%),P:1e-15\n",
            "\t\tRound 2: -35.49 CCCTATTT T:108.0(10.24%),B:154.5(3.52%),P:1e-15\n",
            "\t\t=Final=: -34.04 CCCTATTT T:101.0(10.10%),B:153.9(3.58%),P:1e-14\n",
            "\t\tPerforming exhaustive masking of motif...\n",
            "\t\tReprioritizing potential motifs...\n",
            "\t16 of 25 Initial Sequence: CGCCATTC... (-11.910)\n",
            "\t\tRound 1: -17.95 CGYCATTC T:183.0(16.73%),B:462.5(10.20%),P:1e-7\n",
            "\t\tRound 2: -32.28 SSTCATTY T:216.0(19.44%),B:460.9(10.16%),P:1e-14\n",
            "\t\tRound 3: -32.28 SSTCATTY T:216.0(19.44%),B:460.9(10.16%),P:1e-14\n",
            "\t\t=Final=: -29.33 SSTCATTY T:222.0(22.20%),B:547.0(12.74%),P:1e-12\n",
            "\t\tPerforming exhaustive masking of motif...\n",
            "\t\tReprioritizing potential motifs...\n",
            "\t17 of 25 Initial Sequence: CAGCACTG... (-12.131)\n",
            "\t\tRound 1: -24.78 CAGCACTG T:107.0(10.15%),B:192.4(4.37%),P:1e-10\n",
            "\t\tRound 2: -24.78 CAGCACTG T:107.0(10.15%),B:192.4(4.37%),P:1e-10\n",
            "\t\t=Final=: -27.09 CAGCACTG T:95.0(9.50%),B:161.9(3.77%),P:1e-11\n",
            "\t\tPerforming exhaustive masking of motif...\n",
            "\t\tReprioritizing potential motifs...\n",
            "\t18 of 25 Initial Sequence: TGCCCATC... (-10.012)\n",
            "\t\tRound 1: -6.88 TGCCCATC T:43.0(4.21%),B:101.3(2.32%),P:1e-2\n",
            "\t\tRound 2: -14.57 TVCYCATC T:119.0(11.22%),B:286.9(6.44%),P:1e-6\n",
            "\t\tRound 3: -15.16 TACYCATC T:348.0(29.40%),B:1054.4(21.77%),P:1e-6\n",
            "\t\tRound 4: -16.05 CHCYCATC T:368.0(30.80%),B:1110.9(22.78%),P:1e-6\n",
            "\t\tRound 5: -19.07 TATTCATC T:378.0(31.49%),B:1098.5(22.56%),P:1e-8\n",
            "\t\tRound 6: -20.60 BRTTCATC T:400.0(32.98%),B:1152.9(23.53%),P:1e-8\n",
            "\t\tRound 7: -25.01 YATTCAKC T:291.0(25.26%),B:744.5(15.91%),P:1e-10\n",
            "\t\tRound 8: -25.01 YATTCAKC T:291.0(25.26%),B:744.5(15.91%),P:1e-10\n",
            "\t\t=Final=: -19.87 YATTCAKC T:250.0(25.00%),B:719.4(16.75%),P:1e-8\n",
            "\t\tPerforming exhaustive masking of motif...\n",
            "\t\tReprioritizing potential motifs...\n",
            "\t19 of 25 Initial Sequence: ATCCGATG... (-10.657)\n",
            "\t\tRound 1: -26.78 ATCCGATG T:71.0(6.86%),B:93.4(2.14%),P:1e-11\n",
            "\t\tRound 2: -26.78 ATCCGATG T:71.0(6.86%),B:93.4(2.14%),P:1e-11\n",
            "\t\t=Final=: -21.94 ATCCGATG T:61.0(6.10%),B:89.6(2.09%),P:1e-9\n",
            "\t\tPerforming exhaustive masking of motif...\n",
            "\t\tReprioritizing potential motifs...\n",
            "\t20 of 25 Initial Sequence: GACTCGCA... (-9.318)\n",
            "\t\tRound 1: -2.41 CTCACTGA T:35.0(3.44%),B:112.0(2.57%),P:1e-1\n",
            "\t\tRound 2: -5.55 CTCACTGA T:18.0(1.78%),B:31.2(0.72%),P:1e-2\n",
            "\t\tRound 3: -8.98 CYCACTCA T:26.0(2.57%),B:41.0(0.93%),P:1e-3\n",
            "\t\tRound 4: -12.46 CTCACTCV T:39.0(3.83%),B:62.4(1.43%),P:1e-5\n",
            "\t\tRound 5: -12.46 CTCACTCV T:39.0(3.83%),B:62.4(1.43%),P:1e-5\n",
            "\t\t=Final=: -11.45 CTCACTCV T:37.0(3.70%),B:62.4(1.45%),P:1e-4\n",
            "\t\tPerforming exhaustive masking of motif...\n",
            "\t\tReprioritizing potential motifs...\n",
            "\tRemaining seeds don't look promising (After initial 5 motifs, logp -8.558 > -9.000)\n",
            "\n",
            "\tFinalizing Enrichment Statistics (new in v3.4)\n",
            "\tReading input files...\n",
            "\t5993 total sequences read\n",
            "\tCache length = 11180\n",
            "\tUsing hypergeometric scoring\n",
            "\tChecking enrichment of 20 motif(s)\n",
            "\t|0%                                    50%                                  100%|\n",
            "\t=================================================================================\n",
            "\tOutput in file: output_dir/homerMotifs.motifs8\n",
            "\n",
            "\n",
            "\tScanning input files...\n",
            "\tParsing sequences...\n",
            "\t|0%                                   50%                                  100%|\n",
            "\t================================================================================\n",
            "\tTotal number of Oligos: 467096\n",
            "\tAutoadjustment for sequence coverage in background: 1.00x\n",
            "\n",
            "\tOligos: 467096 of 551887 max\n",
            "\tTree  : 945112 of 2759435 max\n",
            "\tOptimizing memory usage...\n",
            "\tCache length = 11180\n",
            "\tUsing hypergeometric scoring\n",
            "\n",
            "\tGlobal Optimization Phase: Looking for enriched oligos with up to 1 mismatches...\n",
            "\n",
            "\tScreening oligos 467096 (allowing 0 mismatches):\n",
            "\t|0%                                   50%                                  100%|\n",
            "\t================================================================================\n",
            "\t\t67.76% skipped, 32.24% checked (150579 of 467096), of those checked:\n",
            "\t\t67.76% not in target, 0.00% increased p-value, 0.00% high p-value\n",
            "\n",
            "\tScreening oligos 467096 (allowing 1 mismatches):\n",
            "\t|0%                                   50%                                  100%|\n",
            "\t================================================================================\n",
            "\t\t67.76% skipped, 32.24% checked (150579 of 467096), of those checked:\n",
            "\t\t0.00% not in target, 15.32% increased p-value, 9.51% high p-value\n",
            "\tReading input files...\n",
            "\t5993 total sequences read\n",
            "\tCache length = 11180\n",
            "\tUsing hypergeometric scoring\n",
            "\n",
            "\tLocal Optimization Phase:\n",
            "\t1 of 25 Initial Sequence: AAAATAAATG... (-264.074)\n",
            "\t\tRound 1: -466.37 AAAATAAATG T:1139.0(68.00%),B:846.9(17.88%),P:1e-202\n",
            "\t\tRound 2: -469.58 AAAATAAATG T:891.0(58.99%),B:536.0(11.71%),P:1e-203\n",
            "\t\tRound 3: -650.72 NAAATAAAWR T:1136.0(67.91%),B:514.9(11.28%),P:1e-282\n",
            "\t\tRound 4: -702.01 MAAATAAAAH T:1472.0(77.07%),B:730.5(15.63%),P:1e-304\n",
            "\t\tRound 5: -734.08 AAAATGAANN T:1991.0(86.36%),B:1080.1(22.24%),P:1e-318\n",
            "\t\tRound 6: -734.08 AAAATGAANN T:1991.0(86.36%),B:1080.1(22.24%),P:1e-318\n",
            "\t\t=Final=: -694.99 AAAATGAANN T:855.0(85.50%),B:987.4(23.00%),P:1e-301\n",
            "\t\tPerforming exhaustive masking of motif...\n",
            "\t\tReprioritizing potential motifs...\n",
            "\t2 of 25 Initial Sequence: TCCAGCGCTG... (-103.302)\n",
            "\t\tRound 1: -361.14 TCCAGCGCTG T:1033.0(64.42%),B:960.6(20.03%),P:1e-156\n",
            "\t\tRound 2: -407.94 YYCAGCGCTG T:962.0(61.81%),B:747.6(15.97%),P:1e-177\n",
            "\t\tRound 3: -411.08 YYCAGCGCTG T:976.0(62.34%),B:758.9(16.18%),P:1e-178\n",
            "\t\tRound 4: -411.08 YYCAGCGCTG T:976.0(62.34%),B:758.9(16.18%),P:1e-178\n",
            "\t\t=Final=: -295.76 YYCAGCGCTG T:504.0(50.40%),B:576.9(13.43%),P:1e-128\n",
            "\t\tPerforming exhaustive masking of motif...\n",
            "\t\tReprioritizing potential motifs...\n",
            "\t3 of 25 Initial Sequence: TCGCCATTTT... (-95.882)\n",
            "\t\tRound 1: -293.91 TCGCCATTTT T:596.0(44.92%),B:461.8(10.18%),P:1e-127\n",
            "\t\tRound 2: -420.87 TCBCCATTTT T:1352.0(74.15%),B:1233.4(24.96%),P:1e-182\n",
            "\t\tRound 3: -443.36 TCBCCATTTN T:1599.0(79.81%),B:1477.1(29.11%),P:1e-192\n",
            "\t\tRound 4: -454.39 TCBCCATTTT T:1242.0(71.14%),B:1001.2(20.79%),P:1e-197\n",
            "\t\tRound 5: -454.39 TCBCCATTTT T:1242.0(71.14%),B:1001.2(20.79%),P:1e-197\n",
            "\t\t=Final=: -448.73 TCBCCATTTT T:743.0(74.30%),B:1019.0(23.73%),P:1e-194\n",
            "\t\tPerforming exhaustive masking of motif...\n",
            "\t\tReprioritizing potential motifs...\n",
            "\t4 of 25 Initial Sequence: ACTCGAGCTG... (-53.719)\n",
            "\t\tRound 1: -210.74 ACTCGAGCTG T:994.0(63.01%),B:1420.1(28.16%),P:1e-91\n",
            "\t\tRound 2: -277.01 ACTCGAGCTG T:1041.0(64.71%),B:1238.0(25.05%),P:1e-120\n",
            "\t\tRound 3: -308.06 RCTCSAGCTG T:931.0(60.60%),B:954.3(19.92%),P:1e-133\n",
            "\t\tRound 4: -320.55 RCTSSAGCTG T:1085.0(66.23%),B:1160.7(23.67%),P:1e-139\n",
            "\t\tRound 5: -320.55 RCTSSAGCTG T:1085.0(66.23%),B:1160.7(23.67%),P:1e-139\n",
            "\t\t=Final=: -258.77 RCTSSAGCTG T:597.0(59.70%),B:948.2(22.08%),P:1e-112\n",
            "\t\tPerforming exhaustive masking of motif...\n",
            "\t\tReprioritizing potential motifs...\n",
            "\t5 of 25 Initial Sequence: TCAGGCTGGA... (-37.445)\n",
            "\t\tRound 1: -100.08 NCAGGCTGGN T:722.0(51.44%),B:1407.6(27.94%),P:1e-43\n",
            "\t\tRound 2: -139.39 NCAGGCTGGM T:835.0(56.63%),B:1442.6(28.53%),P:1e-60\n",
            "\t\tRound 3: -152.83 NCAGRCTGVV T:1073.0(65.82%),B:1899.9(35.74%),P:1e-66\n",
            "\t\tRound 4: -169.74 NCAGGCTGSM T:939.0(60.92%),B:1508.6(29.62%),P:1e-73\n",
            "\t\tRound 5: -175.68 NSAGRCTGVV T:887.0(58.83%),B:1367.8(27.27%),P:1e-76\n",
            "\t\tRound 6: -175.68 NSAGRCTGVV T:887.0(58.83%),B:1367.8(27.27%),P:1e-76\n",
            "\t\t=Final=: -128.45 NSAGRCTGVV T:504.0(50.40%),B:1040.0(24.22%),P:1e-55\n",
            "\t\tPerforming exhaustive masking of motif...\n",
            "\t\tReprioritizing potential motifs...\n",
            "\t6 of 25 Initial Sequence: GCTGCCGATC... (-32.147)\n",
            "\t\tRound 1: -83.80 GCTGCCGATC T:166.0(15.30%),B:156.2(3.57%),P:1e-36\n",
            "\t\tRound 2: -96.26 GCTGCCGATC T:201.0(18.22%),B:196.5(4.46%),P:1e-41\n",
            "\t\tRound 3: -98.35 GCTGCCGATC T:257.0(22.67%),B:301.8(6.77%),P:1e-42\n",
            "\t\tRound 4: -102.99 GCTGCCGATY T:373.0(31.15%),B:540.2(11.82%),P:1e-44\n",
            "\t\tRound 5: -103.95 GCTGCCGATY T:351.0(29.61%),B:488.2(10.74%),P:1e-45\n",
            "\t\tRound 6: -103.95 GCTGCCGATY T:351.0(29.61%),B:488.2(10.74%),P:1e-45\n",
            "\t\t=Final=: -112.08 GCTGCCGATY T:305.0(30.50%),B:462.7(10.77%),P:1e-48\n",
            "\t\tPerforming exhaustive masking of motif...\n",
            "\t\tReprioritizing potential motifs...\n",
            "\t7 of 25 Initial Sequence: TAGCAATAAA... (-31.300)\n",
            "\t\tRound 1: -76.60 TAGCAATAAA T:162.0(14.96%),B:163.5(3.72%),P:1e-33\n",
            "\t\tRound 2: -112.67 TARCAATAAA T:531.0(41.21%),B:869.7(18.32%),P:1e-48\n",
            "\t\tRound 3: -140.75 TWRVAATAAA T:448.0(36.12%),B:576.0(12.53%),P:1e-61\n",
            "\t\tRound 4: -142.50 TNNVAATVAA T:1114.0(67.19%),B:2056.5(38.05%),P:1e-61\n",
            "\t\tRound 5: -142.50 TNNVAATVAA T:1114.0(67.19%),B:2056.5(38.05%),P:1e-61\n",
            "\t\t=Final=: -152.43 TNNVAATVAA T:681.0(68.10%),B:1633.8(38.05%),P:1e-66\n",
            "\t\tPerforming exhaustive masking of motif...\n",
            "\t\tReprioritizing potential motifs...\n",
            "\t8 of 25 Initial Sequence: AAATCGACTC... (-21.577)\n",
            "\t\tRound 1: -41.00 AAATCGACTC T:86.0(8.24%),B:90.3(2.07%),P:1e-17\n",
            "\t\tRound 2: -48.46 AAATCGACTC T:196.0(17.81%),B:325.1(7.29%),P:1e-21\n",
            "\t\tRound 3: -55.31 AAATCGACTC T:277.0(24.21%),B:505.7(11.10%),P:1e-24\n",
            "\t\tRound 4: -55.31 AAATCGACTC T:277.0(24.21%),B:505.7(11.10%),P:1e-24\n",
            "\t\t=Final=: -55.79 AAATCGACTC T:243.0(24.30%),B:477.0(11.11%),P:1e-24\n",
            "\t\tPerforming exhaustive masking of motif...\n",
            "\t\tReprioritizing potential motifs...\n",
            "\t9 of 25 Initial Sequence: ATCCACTGAA... (-19.747)\n",
            "\t\tRound 1: -34.47 ATCCACTGAA T:87.0(8.34%),B:109.4(2.51%),P:1e-14\n",
            "\t\tRound 2: -60.94 ATCCACTGAA T:371.0(31.01%),B:727.3(15.58%),P:1e-26\n",
            "\t\tRound 3: -74.08 ATCCACTGAA T:482.0(38.26%),B:945.3(19.75%),P:1e-32\n",
            "\t\tRound 4: -77.07 ATCSACTGAA T:445.0(35.93%),B:832.7(17.61%),P:1e-33\n",
            "\t\tRound 5: -87.15 HTCSACTGRR T:598.0(45.03%),B:1171.0(23.87%),P:1e-37\n",
            "\t\tRound 6: -87.15 HTCSACTGRR T:598.0(45.03%),B:1171.0(23.87%),P:1e-37\n",
            "\t\t=Final=: -70.45 HTCSACTGRR T:468.0(46.80%),B:1178.2(27.44%),P:1e-30\n",
            "\t\tPerforming exhaustive masking of motif...\n",
            "\t\tReprioritizing potential motifs...\n",
            "\t10 of 25 Initial Sequence: AACTGAATGC... (-17.019)\n",
            "\t\tRound 1: -35.83 AACTGAATGC T:33.0(3.25%),B:10.1(0.23%),P:1e-15\n",
            "\t\tRound 2: -41.49 CACTGAATGC T:45.0(4.40%),B:20.5(0.46%),P:1e-18\n",
            "\t\tRound 3: -41.85 MACTGAATGM T:47.0(4.59%),B:21.7(0.49%),P:1e-18\n",
            "\t\tRound 4: -45.74 MACTGAATGH T:45.0(4.40%),B:16.6(0.37%),P:1e-19\n",
            "\t\tRound 5: -49.28 MACTGAATGC T:44.0(4.31%),B:12.6(0.28%),P:1e-21\n",
            "\t\tRound 6: -49.28 MACTGAATGC T:44.0(4.31%),B:12.6(0.28%),P:1e-21\n",
            "\t\t=Final=: -46.50 MACTGAATGC T:42.0(4.20%),B:12.6(0.29%),P:1e-20\n",
            "\t\tPerforming exhaustive masking of motif...\n",
            "\t\tReprioritizing potential motifs...\n",
            "\t11 of 25 Initial Sequence: AGAATAAATG... (-16.080)\n",
            "\t\tRound 1: -16.75 AGAATAAATG T:125.0(11.76%),B:288.4(6.49%),P:1e-7\n",
            "\t\tRound 2: -26.26 AVAATAAATG T:216.0(19.44%),B:501.8(11.01%),P:1e-11\n",
            "\t\tRound 3: -34.39 NNAAWAAATG T:155.0(14.37%),B:277.7(6.25%),P:1e-14\n",
            "\t\tRound 4: -34.39 NNAAWAAATG T:155.0(14.37%),B:277.7(6.25%),P:1e-14\n",
            "\t\t=Final=: -33.56 NNAAWAAATG T:137.0(13.70%),B:254.9(5.94%),P:1e-14\n",
            "\t\tPerforming exhaustive masking of motif...\n",
            "\t\tReprioritizing potential motifs...\n",
            "\t12 of 25 Initial Sequence: TGCAAAATAA... (-16.761)\n",
            "\t\tRound 1: -28.58 TGCAAAATAA T:31.0(3.05%),B:14.4(0.33%),P:1e-12\n",
            "\t\tRound 2: -28.58 TGCAAAATAA T:31.0(3.05%),B:14.4(0.33%),P:1e-12\n",
            "\t\t=Final=: -28.10 TGCAAAATAA T:24.0(2.40%),B:6.9(0.16%),P:1e-12\n",
            "\t\tPerforming exhaustive masking of motif...\n",
            "\t\tReprioritizing potential motifs...\n",
            "\t13 of 25 Initial Sequence: GCTGACCACT... (-15.669)\n",
            "\t\tRound 1: -27.89 GCTGACCACT T:29.0(2.86%),B:12.8(0.28%),P:1e-12\n",
            "\t\tRound 2: -27.89 GCTGACCACT T:29.0(2.86%),B:12.8(0.28%),P:1e-12\n",
            "\t\t=Final=: -26.85 GCTGACCACT T:19.0(1.90%),B:2.8(0.07%),P:1e-11\n",
            "\t\tPerforming exhaustive masking of motif...\n",
            "\t\tReprioritizing potential motifs...\n",
            "\t14 of 25 Initial Sequence: AGCTGCGGTC... (-13.201)\n",
            "\t\tRound 1: -23.41 AGCTGCGGTC T:15.0(1.49%),B:0.6(0.00%),P:1e-10\n",
            "\t\tRound 2: -30.59 AGCTGCGVTY T:60.0(5.83%),B:61.9(1.41%),P:1e-13\n",
            "\t\tRound 3: -30.59 AGCTGCGVTY T:60.0(5.83%),B:61.9(1.41%),P:1e-13\n",
            "\t\t=Final=: -31.12 AGCTGCGVTY T:59.0(5.90%),B:61.9(1.44%),P:1e-13\n",
            "\t\tPerforming exhaustive masking of motif...\n",
            "\t\tReprioritizing potential motifs...\n",
            "\t15 of 25 Initial Sequence: AAAATGATGA... (-12.911)\n",
            "\t\tRound 1: -21.66 AAAATGATGA T:27.0(2.67%),B:16.7(0.37%),P:1e-9\n",
            "\t\tRound 2: -23.33 CAAATGATGA T:27.0(2.67%),B:14.0(0.33%),P:1e-10\n",
            "\t\tRound 3: -25.19 CAAATGATGA T:27.0(2.67%),B:12.9(0.28%),P:1e-10\n",
            "\t\tRound 4: -25.19 CAAATGATGA T:27.0(2.67%),B:12.9(0.28%),P:1e-10\n",
            "\t\t=Final=: -24.23 CAAATGATGA T:26.0(2.60%),B:12.9(0.30%),P:1e-10\n",
            "\t\tPerforming exhaustive masking of motif...\n",
            "\t\tReprioritizing potential motifs...\n",
            "\t16 of 25 Initial Sequence: GGATAAAATG... (-12.464)\n",
            "\t\tRound 1: -23.66 GGATAAAATG T:42.0(4.12%),B:40.1(0.93%),P:1e-10\n",
            "\t\tRound 2: -27.87 GGATAAAATG T:64.0(6.20%),B:76.3(1.75%),P:1e-12\n",
            "\t\tRound 3: -27.87 GGATAAAATG T:64.0(6.20%),B:76.3(1.75%),P:1e-12\n",
            "\t\t=Final=: -26.05 GGATAAAATG T:60.0(6.00%),B:75.1(1.75%),P:1e-11\n",
            "\t\tPerforming exhaustive masking of motif...\n",
            "\t\tReprioritizing potential motifs...\n",
            "\t17 of 25 Initial Sequence: ATACGCGAGC... (-12.263)\n",
            "\t\tRound 1: -23.69 ATACGCGAGC T:18.0(1.78%),B:3.5(0.07%),P:1e-10\n",
            "\t\tRound 2: -23.69 ATACGCGAGC T:18.0(1.78%),B:3.5(0.07%),P:1e-10\n",
            "\t\t=Final=: -23.58 ATACGCGAGC T:20.0(2.00%),B:5.4(0.13%),P:1e-10\n",
            "\t\tPerforming exhaustive masking of motif...\n",
            "\t\tReprioritizing potential motifs...\n",
            "\t18 of 25 Initial Sequence: GCTCACACTC... (-12.263)\n",
            "\t\tRound 1: -23.33 GCTCACACTC T:111.0(10.51%),B:211.1(4.80%),P:1e-10\n",
            "\t\tRound 2: -26.06 GCTCACACTC T:73.0(7.04%),B:102.0(2.32%),P:1e-11\n",
            "\t\tRound 3: -31.11 GCTCACACTC T:68.0(6.58%),B:75.8(1.73%),P:1e-13\n",
            "\t\tRound 4: -31.11 GCTCACACTC T:68.0(6.58%),B:75.8(1.73%),P:1e-13\n",
            "\t\t=Final=: -33.56 GCTCACACTC T:68.0(6.80%),B:75.8(1.76%),P:1e-14\n",
            "\t\tPerforming exhaustive masking of motif...\n",
            "\t\tReprioritizing potential motifs...\n",
            "\t19 of 25 Initial Sequence: TGCAAATCAC... (-11.405)\n",
            "\t\tRound 1: -23.10 TGCAAATCAC T:69.0(6.67%),B:100.2(2.30%),P:1e-10\n",
            "\t\tRound 2: -25.45 TGCAAATCAC T:30.0(2.96%),B:16.4(0.37%),P:1e-11\n",
            "\t\tRound 3: -25.45 TGCAAATCAC T:30.0(2.96%),B:16.4(0.37%),P:1e-11\n",
            "\t\t=Final=: -25.87 TGCAAATCAC T:30.0(3.00%),B:16.4(0.38%),P:1e-11\n",
            "\t\tPerforming exhaustive masking of motif...\n",
            "\t\tReprioritizing potential motifs...\n",
            "\t20 of 25 Initial Sequence: ACTCAAATGC... (-11.057)\n",
            "\t\tRound 1: -25.05 ACTCAAATGC T:22.0(2.18%),B:6.1(0.14%),P:1e-10\n",
            "\t\tRound 2: -28.10 ACTCAAATGC T:25.0(2.47%),B:7.9(0.16%),P:1e-12\n",
            "\t\tRound 3: -28.10 ACTCAAATGC T:25.0(2.47%),B:7.9(0.16%),P:1e-12\n",
            "\t\t=Final=: -28.25 ACTCAAATGC T:25.0(2.50%),B:7.9(0.18%),P:1e-12\n",
            "\t\tPerforming exhaustive masking of motif...\n",
            "\t\tReprioritizing potential motifs...\n",
            "\t21 of 25 Initial Sequence: GCTCAGCTGA... (-10.971)\n",
            "\t\tRound 1: -28.02 GCTCAGCTGA T:23.0(2.27%),B:5.7(0.12%),P:1e-12\n",
            "\t\tRound 2: -28.02 GCTCAGCTGA T:23.0(2.27%),B:5.7(0.12%),P:1e-12\n",
            "\t\t=Final=: -26.50 GCTCAGCTGA T:21.0(2.10%),B:4.5(0.10%),P:1e-11\n",
            "\t\tPerforming exhaustive masking of motif...\n",
            "\t\tReprioritizing potential motifs...\n",
            "\t22 of 25 Initial Sequence: TCAGCCACTC... (-10.873)\n",
            "\t\tRound 1: -21.15 TCAGCCACTC T:21.0(2.08%),B:8.5(0.19%),P:1e-9\n",
            "\t\tRound 2: -23.77 TCAGCCACTC T:38.0(3.73%),B:32.8(0.74%),P:1e-10\n",
            "\t\tRound 3: -24.26 YCAGCCAYTC T:34.0(3.34%),B:24.0(0.56%),P:1e-10\n",
            "\t\tRound 4: -28.54 TCAGCCAYTC T:45.0(4.40%),B:37.4(0.86%),P:1e-12\n",
            "\t\tRound 5: -30.66 TCAGCCAYTC T:34.0(3.34%),B:16.6(0.37%),P:1e-13\n",
            "\t\tRound 6: -30.66 TCAGCCAYTC T:34.0(3.34%),B:16.6(0.37%),P:1e-13\n",
            "\t\t=Final=: -29.73 TCAGCCAYTC T:33.0(3.30%),B:16.8(0.39%),P:1e-12\n",
            "\t\tPerforming exhaustive masking of motif...\n",
            "\t\tReprioritizing potential motifs...\n",
            "\t23 of 25 Initial Sequence: GATATTTATC... (-10.669)\n",
            "\t\tRound 1: -23.52 GATATTTATC T:54.0(5.26%),B:64.0(1.48%),P:1e-10\n",
            "\t\tRound 2: -27.53 GHTATTTATC T:88.0(8.43%),B:132.4(3.03%),P:1e-11\n",
            "\t\tRound 3: -32.90 GHKATTTATC T:129.0(12.11%),B:217.5(4.93%),P:1e-14\n",
            "\t\tRound 4: -32.90 GHKATTTATC T:129.0(12.11%),B:217.5(4.93%),P:1e-14\n",
            "\t\t=Final=: -32.59 GHKATTTATC T:122.0(12.20%),B:215.4(5.02%),P:1e-14\n",
            "\t\tPerforming exhaustive masking of motif...\n",
            "\t\tReprioritizing potential motifs...\n",
            "\t24 of 25 Initial Sequence: TCGCTGAATG... (-10.267)\n",
            "\t\tRound 1: -19.77 NCGCTGAATN T:78.0(7.51%),B:137.8(3.14%),P:1e-8\n",
            "\t\tRound 2: -21.78 KCGCTGAATR T:60.0(5.83%),B:83.6(1.91%),P:1e-9\n",
            "\t\tRound 3: -27.84 TMGCTGAATA T:49.0(4.78%),B:44.5(1.02%),P:1e-12\n",
            "\t\tRound 4: -27.84 TMGCTGAATA T:49.0(4.78%),B:44.5(1.02%),P:1e-12\n",
            "\t\t=Final=: -30.50 TMGCTGAATA T:50.0(5.00%),B:44.5(1.04%),P:1e-13\n",
            "\t\tPerforming exhaustive masking of motif...\n",
            "\t\tReprioritizing potential motifs...\n",
            "\t25 of 25 Initial Sequence: CATTCAGTTT... (-10.012)\n",
            "\t\tRound 1: -2.32 CATTCAGTTT T:36.0(3.54%),B:117.5(2.69%),P:1e-1\n",
            "\t\tRound 2: -5.31 CATTCAGTTT T:12.0(1.19%),B:16.7(0.37%),P:1e-2\n",
            "\t\tRound 3: -15.24 BAWTCAGTKT T:89.0(8.52%),B:190.9(4.33%),P:1e-6\n",
            "\t\tRound 4: -15.24 BAWTCAGTKT T:89.0(8.52%),B:190.9(4.33%),P:1e-6\n",
            "\t\t=Final=: -16.19 BAWTCAGTKT T:87.0(8.70%),B:186.1(4.33%),P:1e-7\n",
            "\t\tPerforming exhaustive masking of motif...\n",
            "\t\tReprioritizing potential motifs...\n",
            "\n",
            "\tFinalizing Enrichment Statistics (new in v3.4)\n",
            "\tReading input files...\n",
            "\t5993 total sequences read\n",
            "\tCache length = 11180\n",
            "\tUsing hypergeometric scoring\n",
            "\tChecking enrichment of 25 motif(s)\n",
            "\t|0%                                    50%                                  100%|\n",
            "\t=================================================================================\n",
            "\tOutput in file: output_dir/homerMotifs.motifs10\n",
            "\n",
            "\n",
            "\t-blen automatically set to 2\n",
            "\tScanning input files...\n",
            "\tParsing sequences...\n",
            "\t|0%                                   50%                                  100%|\n",
            "\t================================================================================\n",
            "\tTotal number of Oligos: 1275813\n",
            "\tAutoadjustment for sequence coverage in background: 1.00x\n",
            "\n",
            "\tOligos: 1275813 of 1426334 max\n",
            "\tTree  : 3302204 of 7131670 max\n",
            "\tOptimizing memory usage...\n",
            "\tCache length = 11180\n",
            "\tUsing hypergeometric scoring\n",
            "\n",
            "\tGlobal Optimization Phase: Looking for enriched oligos with up to 1 mismatches...\n",
            "\n",
            "\tScreening oligos 1275813 (allowing 0 mismatches):\n",
            "\t|0%                                   50%                                  100%|\n",
            "\t================================================================================\n",
            "\t\t82.65% skipped, 17.35% checked (221360 of 1275813), of those checked:\n",
            "\t\t82.65% not in target, 0.00% increased p-value, 0.00% high p-value\n",
            "\n",
            "\tScreening oligos 1275813 (allowing 1 mismatches):\n",
            "\t|0%                                   50%                                  100%|\n",
            "\t===============================================================================\n",
            "\t\t82.65% skipped, 17.35% checked (221360 of 1275813), of those checked:\n",
            "\t\t0.00% not in target, 10.12% increased p-value, 0.49% high p-value\n",
            "\tReading input files...\n",
            "\t5993 total sequences read\n",
            "\tCache length = 11180\n",
            "\tUsing hypergeometric scoring\n",
            "\n",
            "\tLocal Optimization Phase:\n",
            "\t1 of 25 Initial Sequence: TTTATTTATTTT... (-112.955)\n",
            "\t\tRound 1: -386.73 TTTATTTATTTT T:1294.0(72.60%),B:1261.3(25.45%),P:1e-167\n",
            "\t\tRound 2: -396.24 TTTATTTATTTT T:846.0(57.11%),B:616.4(13.36%),P:1e-172\n",
            "\t\tRound 3: -420.86 TTTATTTWTTTT T:1294.0(72.60%),B:1159.2(23.66%),P:1e-182\n",
            "\t\tRound 4: -428.56 TTYATTTWTTTT T:1298.0(72.71%),B:1142.8(23.35%),P:1e-186\n",
            "\t\tRound 5: -439.11 TTYATTTWTTTT T:1295.0(72.63%),B:1108.9(22.74%),P:1e-190\n",
            "\t\tRound 6: -439.11 TTYATTTWTTTT T:1295.0(72.63%),B:1108.9(22.74%),P:1e-190\n",
            "\t\t=Final=: -375.10 TTYATTTWTTTT T:617.0(61.70%),B:745.9(17.37%),P:1e-162\n",
            "\t\tPerforming exhaustive masking of motif...\n",
            "\t\tReprioritizing potential motifs...\n",
            "\t2 of 25 Initial Sequence: AAATGAAAAATG... (-59.770)\n",
            "\t\tRound 1: -220.43 AAATGAAAAATG T:473.0(37.70%),B:416.8(9.23%),P:1e-95\n",
            "\t\tRound 2: -319.50 AAATGAAAAATG T:792.0(54.72%),B:705.0(15.12%),P:1e-138\n",
            "\t\tRound 3: -377.22 AAATGAARAHTV T:1036.0(64.53%),B:922.6(19.32%),P:1e-163\n",
            "\t\tRound 4: -392.73 AAATGAARAHTV T:1414.0(75.70%),B:1407.3(27.94%),P:1e-170\n",
            "\t\tRound 5: -409.71 AAATGAARAHWN T:1139.0(68.00%),B:982.2(20.44%),P:1e-177\n",
            "\t\tRound 6: -414.87 AAATGAARAHTV T:1044.0(64.81%),B:841.5(17.79%),P:1e-180\n",
            "\t\tRound 7: -414.87 AAATGAARAHTV T:1044.0(64.81%),B:841.5(17.79%),P:1e-180\n",
            "\t\t=Final=: -402.82 AAATGAARAHTV T:636.0(63.60%),B:749.2(17.45%),P:1e-174\n",
            "\t\tPerforming exhaustive masking of motif...\n",
            "\t\tReprioritizing potential motifs...\n",
            "\t3 of 25 Initial Sequence: AAATAAACTGCT... (-38.848)\n",
            "\t\tRound 1: -114.62 AAATAAACTGCT T:314.0(26.96%),B:374.0(8.34%),P:1e-49\n",
            "\t\tRound 2: -171.66 AAATAAACTGCT T:499.0(39.30%),B:585.1(12.74%),P:1e-74\n",
            "\t\tRound 3: -216.83 AAATAAAMTGCH T:823.0(56.11%),B:1064.7(21.95%),P:1e-94\n",
            "\t\tRound 4: -248.25 VAMTRAAVTBVN T:1124.0(67.52%),B:1498.7(29.45%),P:1e-107\n",
            "\t\tRound 5: -272.26 AAMTGAANTBVN T:1483.0(77.32%),B:2007.6(37.34%),P:1e-118\n",
            "\t\tRound 6: -299.27 VRMTSVANHBNN T:1609.0(80.01%),B:2073.7(38.29%),P:1e-129\n",
            "\t\tRound 7: -309.68 AAMTGAADTTVA T:1469.0(77.00%),B:1807.9(34.35%),P:1e-134\n",
            "\t\tRound 8: -329.11 AAMTGAADYTVD T:1439.0(76.30%),B:1678.7(32.35%),P:1e-142\n",
            "\t\tRound 9: -330.38 AAMTGRADYKVD T:1505.0(77.81%),B:1771.4(33.80%),P:1e-143\n",
            "\t\tRound 10: -333.21 AACTGVADYKVN T:1483.0(77.32%),B:1726.7(33.10%),P:1e-144\n",
            "\t\t=Final=: -327.63 AACTGVADYKVN T:737.0(73.70%),B:1284.7(29.92%),P:1e-142\n",
            "\t\tPerforming exhaustive masking of motif...\n",
            "\t\tReprioritizing potential motifs...\n",
            "\t4 of 25 Initial Sequence: GCCCAGCGCTGA... (-28.443)\n",
            "\t\tRound 1: -246.31 GCCCAGCGCTGA T:703.0(50.51%),B:749.0(15.99%),P:1e-106\n",
            "\t\tRound 2: -336.30 SCCCAGCGCTGA T:839.0(56.80%),B:736.5(15.75%),P:1e-146\n",
            "\t\tRound 3: -340.99 NBYCAGCGCTGR T:751.0(52.83%),B:595.7(12.94%),P:1e-148\n",
            "\t\tRound 4: -342.90 NBYCAGCGCTGR T:887.0(58.83%),B:791.7(16.82%),P:1e-148\n",
            "\t\tRound 5: -342.90 NBYCAGCGCTGR T:887.0(58.83%),B:791.7(16.82%),P:1e-148\n",
            "\t\t=Final=: -238.11 NBYCAGCGCTGR T:440.0(44.00%),B:521.6(12.15%),P:1e-103\n",
            "\t\tPerforming exhaustive masking of motif...\n",
            "\t\tReprioritizing potential motifs...\n",
            "\t5 of 25 Initial Sequence: GAAATAAACTGC... (-25.084)\n",
            "\t\tRound 1: -73.23 GAAATAAACTGC T:199.0(18.05%),B:247.6(5.59%),P:1e-31\n",
            "\t\tRound 2: -89.64 GAAATAAACTGC T:205.0(18.54%),B:218.7(4.95%),P:1e-38\n",
            "\t\tRound 3: -102.85 DAAATAAACTGC T:174.0(15.98%),B:134.2(3.07%),P:1e-44\n",
            "\t\tRound 4: -120.79 NAAATAAAVTGC T:157.0(14.54%),B:82.3(1.89%),P:1e-52\n",
            "\t\tRound 5: -147.33 NRAATAAAMTKC T:562.0(43.01%),B:795.5(16.90%),P:1e-63\n",
            "\t\tRound 6: -179.67 NVAATAAANNNN T:772.0(53.81%),B:1110.0(22.76%),P:1e-78\n",
            "\t\tRound 7: -190.92 NVAATAAANNBH T:631.0(46.81%),B:782.9(16.65%),P:1e-82\n",
            "\t\tRound 8: -190.92 NVAATAAANNBH T:631.0(46.81%),B:782.9(16.65%),P:1e-82\n",
            "\t\t=Final=: -205.27 NVAATAAANNBH T:524.0(52.40%),B:852.6(19.85%),P:1e-89\n",
            "\t\tPerforming exhaustive masking of motif...\n",
            "\t\tReprioritizing potential motifs...\n",
            "\t6 of 25 Initial Sequence: AGGCTGCCCATT... (-20.890)\n",
            "\t\tRound 1: -106.59 AGGCTGCCCATT T:293.0(25.41%),B:355.5(7.93%),P:1e-46\n",
            "\t\tRound 2: -174.75 NGGCTGCCCATY T:1041.0(64.71%),B:1698.8(32.66%),P:1e-75\n",
            "\t\tRound 3: -227.91 HGGCTGCCCATY T:562.0(43.01%),B:550.8(12.02%),P:1e-98\n",
            "\t\tRound 4: -255.57 HRRCTGSCSATY T:790.0(54.63%),B:872.4(18.38%),P:1e-110\n",
            "\t\tRound 5: -281.46 MRRCTGCCCATY T:728.0(51.73%),B:696.2(14.96%),P:1e-122\n",
            "\t\tRound 6: -304.66 MRRMTGSSSAKY T:764.0(53.44%),B:696.8(14.96%),P:1e-132\n",
            "\t\tRound 7: -304.66 MRRMTGSSSAKY T:764.0(53.44%),B:696.8(14.96%),P:1e-132\n",
            "\t\t=Final=: -240.18 MRRMTGSSSAKY T:464.0(46.40%),B:583.0(13.58%),P:1e-104\n",
            "\t\tPerforming exhaustive masking of motif...\n",
            "\t\tReprioritizing potential motifs...\n",
            "\t7 of 25 Initial Sequence: GAAAATGCGCTA... (-17.677)\n",
            "\t\tRound 1: -35.83 GAAAATGCGCTA T:33.0(3.25%),B:10.9(0.23%),P:1e-15\n",
            "\t\tRound 2: -53.09 SAAAATGSGSTM T:56.0(5.45%),B:22.7(0.51%),P:1e-23\n",
            "\t\tRound 3: -70.52 VAAAATGSGVYM T:254.0(22.44%),B:382.5(8.51%),P:1e-30\n",
            "\t\tRound 4: -93.03 VAAAATGSBNNM T:336.0(28.55%),B:492.2(10.83%),P:1e-40\n",
            "\t\tRound 5: -107.51 VAAAATGSBNNM T:408.0(33.52%),B:604.5(13.12%),P:1e-46\n",
            "\t\tRound 6: -115.07 VAAAATGSGVBM T:480.0(38.14%),B:739.7(15.81%),P:1e-49\n",
            "\t\tRound 7: -118.82 VAAAATGVSNNM T:681.0(49.41%),B:1195.5(24.29%),P:1e-51\n",
            "\t\tRound 8: -131.45 VAAAATGVSNNM T:685.0(49.61%),B:1139.6(23.30%),P:1e-57\n",
            "\t\tRound 9: -138.39 NAAAATGVGNSM T:367.0(30.73%),B:418.0(9.25%),P:1e-60\n",
            "\t\tRound 10: -139.84 NAAAATGRGNBM T:356.0(29.97%),B:391.9(8.70%),P:1e-60\n",
            "\t\t=Final=: -157.62 NAAAATGRGNBM T:386.0(38.60%),B:563.1(13.11%),P:1e-68\n",
            "\t\tPerforming exhaustive masking of motif...\n",
            "\t\tReprioritizing potential motifs...\n",
            "\t8 of 25 Initial Sequence: GTCATTCGAATG... (-16.702)\n",
            "\t\tRound 1: -37.73 GTCATTCGAATG T:140.0(13.07%),B:224.7(5.08%),P:1e-16\n",
            "\t\tRound 2: -51.59 GTCATTCGAATG T:178.0(16.31%),B:269.4(6.07%),P:1e-22\n",
            "\t\tRound 3: -61.81 BTCATTCGAATG T:177.0(16.23%),B:233.6(5.28%),P:1e-26\n",
            "\t\tRound 4: -62.61 KTCATTCGAATG T:112.0(10.60%),B:98.4(2.26%),P:1e-27\n",
            "\t\tRound 5: -62.61 KTCATTCGAATG T:112.0(10.60%),B:98.4(2.26%),P:1e-27\n",
            "\t\t=Final=: -52.27 KTCATTCGAATG T:90.0(9.00%),B:83.8(1.95%),P:1e-22\n",
            "\t\tPerforming exhaustive masking of motif...\n",
            "\t\tReprioritizing potential motifs...\n",
            "\t9 of 25 Initial Sequence: AATAAAATAAAA... (-15.028)\n",
            "\t\tRound 1: -8.83 AATAAAATAAAA T:11.0(1.09%),B:7.5(0.16%),P:1e-3\n",
            "\t\tRound 2: -9.91 AATAMAATAAAA T:9.0(0.90%),B:3.7(0.07%),P:1e-4\n",
            "\t\tRound 3: -9.91 AATAMAATAAAA T:9.0(0.90%),B:3.7(0.07%),P:1e-4\n",
            "\t\t=Final=: -10.19 AATAMAATAAAA T:9.0(0.90%),B:3.7(0.09%),P:1e-4\n",
            "\t\tPerforming exhaustive masking of motif...\n",
            "\t\tReprioritizing potential motifs...\n",
            "\t10 of 25 Initial Sequence: AGCTGCAGGCTG... (-15.028)\n",
            "\t\tRound 1: -52.80 AGCTGCAGGCTG T:178.0(16.31%),B:264.9(5.96%),P:1e-22\n",
            "\t\tRound 2: -65.48 AGCTGCAGGCTG T:361.0(30.31%),B:675.7(14.55%),P:1e-28\n",
            "\t\tRound 3: -78.19 AGCTGCAGGCTG T:311.0(26.74%),B:489.9(10.76%),P:1e-33\n",
            "\t\tRound 4: -85.84 AGCTGCAGRCTG T:273.0(23.90%),B:374.5(8.34%),P:1e-37\n",
            "\t\tRound 5: -90.89 AGCTGCAGDCTG T:249.0(22.05%),B:305.7(6.86%),P:1e-39\n",
            "\t\tRound 6: -90.89 AGCTGCAGDCTG T:249.0(22.05%),B:305.7(6.86%),P:1e-39\n",
            "\t\t=Final=: -88.53 AGCTGCAGDCTG T:228.0(22.80%),B:321.5(7.49%),P:1e-38\n",
            "\t\tPerforming exhaustive masking of motif...\n",
            "\t\tReprioritizing potential motifs...\n",
            "\t11 of 25 Initial Sequence: AAAATGAAAATG... (-11.683)\n",
            "\t\tRound 1: -13.62 AAAATGAAAATG T:14.0(1.39%),B:6.9(0.14%),P:1e-5\n",
            "\t\tRound 2: -20.81 ARAHTGARAATG T:104.0(9.88%),B:201.4(4.57%),P:1e-9\n",
            "\t\tRound 3: -24.46 ARACTSAAAATG T:37.0(3.63%),B:29.6(0.67%),P:1e-10\n",
            "\t\tRound 4: -29.18 ARRCTGWAAATG T:75.0(7.23%),B:97.8(2.23%),P:1e-12\n",
            "\t\tRound 5: -32.54 ARRCTGWAAATG T:184.0(16.81%),B:368.4(8.21%),P:1e-14\n",
            "\t\tRound 6: -38.09 ARRCTGARAATG T:183.0(16.73%),B:336.2(7.53%),P:1e-16\n",
            "\t\tRound 7: -48.71 ANRCTGAAAATR T:249.0(22.05%),B:461.9(10.18%),P:1e-21\n",
            "\t\tRound 8: -48.71 ANRCTGAAAATR T:249.0(22.05%),B:461.9(10.18%),P:1e-21\n",
            "\t\t=Final=: -43.78 ANRCTGAAAATR T:171.0(17.10%),B:312.0(7.27%),P:1e-19\n",
            "\t\tPerforming exhaustive masking of motif...\n",
            "\t\tReprioritizing potential motifs...\n",
            "\t12 of 25 Initial Sequence: CAAATAAAAATG... (-11.683)\n",
            "\t\tRound 1: -6.72 CAAATAANAATG T:6.0(0.60%),B:2.6(0.05%),P:1e-2\n",
            "\t\tRound 2: -11.23 CAAATAANACTG T:17.0(1.69%),B:14.6(0.33%),P:1e-4\n",
            "\t\tRound 3: -17.68 SAAATAAYAMTG T:13.0(1.29%),B:2.8(0.05%),P:1e-7\n",
            "\t\tRound 4: -27.37 SAAATAAYACTG T:19.0(1.88%),B:1.7(0.02%),P:1e-11\n",
            "\t\tRound 5: -27.37 SAAATAAYACTG T:19.0(1.88%),B:1.7(0.02%),P:1e-11\n",
            "\t\t=Final=: -25.75 SAAATAAYACTG T:17.0(1.70%),B:1.7(0.04%),P:1e-11\n",
            "\t\tPerforming exhaustive masking of motif...\n",
            "\t\tReprioritizing potential motifs...\n",
            "\t13 of 25 Initial Sequence: TAAAATAAAATG... (-11.683)\n",
            "\t\tRound 1: -7.88 TAAAATAAAATG T:9.0(0.90%),B:5.4(0.12%),P:1e-3\n",
            "\t\tRound 2: -14.49 WAAAATAAAMTG T:11.0(1.09%),B:2.0(0.02%),P:1e-6\n",
            "\t\tRound 3: -14.49 WAAAATAAAMTG T:11.0(1.09%),B:2.0(0.02%),P:1e-6\n",
            "\t\t=Final=: -14.40 WAAAATAAAMTG T:11.0(1.10%),B:3.0(0.07%),P:1e-6\n",
            "\t\tPerforming exhaustive masking of motif...\n",
            "\t\tReprioritizing potential motifs...\n",
            "\t14 of 25 Initial Sequence: TCAGCTGCGATT... (-11.341)\n",
            "\t\tRound 1: -20.89 TCAGCTGCGATT T:15.0(1.49%),B:2.4(0.05%),P:1e-9\n",
            "\t\tRound 2: -29.34 TCAGYTGCGATT T:33.0(3.25%),B:16.6(0.37%),P:1e-12\n",
            "\t\tRound 3: -34.90 TCAGCTKHGABT T:44.0(4.31%),B:26.0(0.60%),P:1e-15\n",
            "\t\tRound 4: -34.90 TCAGCTKHGABT T:44.0(4.31%),B:26.0(0.60%),P:1e-15\n",
            "\t\t=Final=: -35.05 TCAGCTKHGABT T:45.0(4.50%),B:28.6(0.67%),P:1e-15\n",
            "\t\tPerforming exhaustive masking of motif...\n",
            "\t\tReprioritizing potential motifs...\n",
            "\t15 of 25 Initial Sequence: CTAAAATAACTG... (-11.341)\n",
            "\t\tRound 1: -21.15 CTAAAATAACTG T:21.0(2.08%),B:8.5(0.19%),P:1e-9\n",
            "\t\tRound 2: -21.15 CTAAAATAACTG T:21.0(2.08%),B:8.5(0.19%),P:1e-9\n",
            "\t\t=Final=: -21.46 CTAAAATAACTG T:21.0(2.10%),B:8.5(0.20%),P:1e-9\n",
            "\t\tPerforming exhaustive masking of motif...\n",
            "\t\tReprioritizing potential motifs...\n",
            "\t16 of 25 Initial Sequence: TGGAGATCTGTA... (-11.341)\n",
            "\t\tRound 1: -17.37 TGGAGATCTGTA T:21.0(2.08%),B:12.1(0.28%),P:1e-7\n",
            "\t\tRound 2: -17.37 TGGAGATCTGTA T:21.0(2.08%),B:12.1(0.28%),P:1e-7\n",
            "\t\t=Final=: -17.05 TGGAGATCTGTA T:21.0(2.10%),B:13.1(0.31%),P:1e-7\n",
            "\t\tPerforming exhaustive masking of motif...\n",
            "\t\tReprioritizing potential motifs...\n",
            "\t17 of 25 Initial Sequence: TCCAGCCTAAAT... (-10.012)\n",
            "\t\tRound 1: -22.00 TCCAGCCTAAAT T:19.0(1.88%),B:5.1(0.12%),P:1e-9\n",
            "\t\tRound 2: -32.62 TCCAGCCTAAAT T:26.0(2.57%),B:5.7(0.12%),P:1e-14\n",
            "\t\tRound 3: -32.62 TCCAGCCTAAAT T:26.0(2.57%),B:5.7(0.12%),P:1e-14\n",
            "\t\t=Final=: -32.54 TCCAGCCTAAAT T:27.0(2.70%),B:6.6(0.15%),P:1e-14\n",
            "\t\tPerforming exhaustive masking of motif...\n",
            "\t\tReprioritizing potential motifs...\n",
            "\t18 of 25 Initial Sequence: CACCCACTCTAG... (-10.012)\n",
            "\t\tRound 1: -15.03 CACCCACTCTAG T:10.0(1.00%),B:0.7(0.00%),P:1e-6\n",
            "\t\tRound 2: -36.96 CACCCACTCTNG T:178.0(16.31%),B:329.2(7.38%),P:1e-16\n",
            "\t\tRound 3: -43.09 CACCCACTCTNG T:149.0(13.85%),B:227.9(5.15%),P:1e-18\n",
            "\t\tRound 4: -48.60 CACCCACTCTNS T:255.0(22.52%),B:479.3(10.56%),P:1e-21\n",
            "\t\tRound 5: -55.17 CWCCCACTCYNS T:201.0(18.22%),B:311.4(6.99%),P:1e-23\n",
            "\t\tRound 6: -55.17 CWCCCACTCYNS T:201.0(18.22%),B:311.4(6.99%),P:1e-23\n",
            "\t\t=Final=: -59.01 CWCCCACTCYNS T:186.0(18.60%),B:298.8(6.96%),P:1e-25\n",
            "\t\tPerforming exhaustive masking of motif...\n",
            "\t\tReprioritizing potential motifs...\n",
            "\t19 of 25 Initial Sequence: CACCGGCTGCTG... (-10.012)\n",
            "\t\tRound 1: -25.74 CACCGGCTGCTG T:107.0(10.15%),B:187.1(4.26%),P:1e-11\n",
            "\t\tRound 2: -29.55 CACCGGCTGCTG T:83.0(7.97%),B:113.6(2.60%),P:1e-12\n",
            "\t\tRound 3: -35.49 CACCGGCTGCTG T:86.0(8.24%),B:104.9(2.39%),P:1e-15\n",
            "\t\tRound 4: -36.35 CACCRGCTGCTG T:166.0(15.30%),B:300.7(6.75%),P:1e-15\n",
            "\t\tRound 5: -36.57 CACCSGCTGCKG T:85.0(8.15%),B:99.7(2.28%),P:1e-15\n",
            "\t\tRound 6: -40.78 CACCSGCTGCKG T:83.0(7.97%),B:84.1(1.94%),P:1e-17\n",
            "\t\tRound 7: -41.32 CACMVGCTGCKS T:84.0(8.06%),B:85.6(1.96%),P:1e-17\n",
            "\t\tRound 8: -41.32 CACMVGCTGCKS T:84.0(8.06%),B:85.6(1.96%),P:1e-17\n",
            "\t\t=Final=: -41.86 CACMVGCTGCKS T:81.0(8.10%),B:85.3(1.99%),P:1e-18\n",
            "\t\tPerforming exhaustive masking of motif...\n",
            "\t\tReprioritizing potential motifs...\n",
            "\t20 of 25 Initial Sequence: ACCCCGCTGAAT... (-10.012)\n",
            "\t\tRound 1: -19.23 ACCCCGCTGAAT T:18.0(1.78%),B:6.3(0.14%),P:1e-8\n",
            "\t\tRound 2: -19.23 ACCCCGCTGAAT T:18.0(1.78%),B:6.3(0.14%),P:1e-8\n",
            "\t\t=Final=: -19.48 ACCCCGCTGAAT T:18.0(1.80%),B:6.3(0.15%),P:1e-8\n",
            "\t\tPerforming exhaustive masking of motif...\n",
            "\t\tReprioritizing potential motifs...\n",
            "\t21 of 25 Initial Sequence: AAGCTGAATGAA... (-10.012)\n",
            "\t\tRound 1: -33.91 AAGCTGAATGAA T:23.0(2.27%),B:2.1(0.05%),P:1e-14\n",
            "\t\tRound 2: -37.46 AAGCTGAATGAA T:35.0(3.44%),B:11.7(0.26%),P:1e-16\n",
            "\t\tRound 3: -38.85 AAGCTGAATGAA T:26.0(2.57%),B:2.9(0.05%),P:1e-16\n",
            "\t\tRound 4: -39.10 AAGCTGAATGAA T:89.0(8.52%),B:103.0(2.35%),P:1e-16\n",
            "\t\tRound 5: -48.77 AARCTGAATGAA T:132.0(12.37%),B:169.0(3.86%),P:1e-21\n",
            "\t\tRound 6: -48.77 AARCTGAATGAA T:132.0(12.37%),B:169.0(3.86%),P:1e-21\n",
            "\t\t=Final=: -43.01 AARCTGAATGAA T:120.0(12.00%),B:174.6(4.07%),P:1e-18\n",
            "\t\tPerforming exhaustive masking of motif...\n",
            "\t\tReprioritizing potential motifs...\n",
            "\t22 of 25 Initial Sequence: TCCCATCGGATG... (-10.012)\n",
            "\t\tRound 1: -33.23 TCCCATCGGATG T:24.0(2.37%),B:3.5(0.07%),P:1e-14\n",
            "\t\tRound 2: -36.96 TCCCATCGGATG T:51.0(4.97%),B:33.8(0.77%),P:1e-16\n",
            "\t\tRound 3: -48.23 TCYCATCSGATG T:62.0(6.01%),B:36.3(0.83%),P:1e-20\n",
            "\t\tRound 4: -48.23 TCYCATCSGATG T:62.0(6.01%),B:36.3(0.83%),P:1e-20\n",
            "\t\t=Final=: -34.21 TCYCATCSGATG T:49.0(4.90%),B:36.3(0.85%),P:1e-14\n",
            "\t\tPerforming exhaustive masking of motif...\n",
            "\t\tReprioritizing potential motifs...\n",
            "\t23 of 25 Initial Sequence: AAAATAAATAAC... (-10.012)\n",
            "\t\tRound 1: -5.85 TBKAYAAATAVC T:9.0(0.90%),B:8.2(0.19%),P:1e-2\n",
            "\t\tRound 2: -6.97 TCTACAAATAAC T:10.0(1.00%),B:8.8(0.19%),P:1e-3\n",
            "\t\tRound 3: -19.40 TYTACAAATAAC T:41.0(4.02%),B:47.2(1.09%),P:1e-8\n",
            "\t\tRound 4: -22.62 THTACAAATAAC T:40.0(3.92%),B:38.7(0.88%),P:1e-9\n",
            "\t\tRound 5: -22.62 THTACAAATAAC T:40.0(3.92%),B:38.7(0.88%),P:1e-9\n",
            "\t\t=Final=: -25.76 THTACAAATAAC T:43.0(4.30%),B:39.2(0.91%),P:1e-11\n",
            "\t\tPerforming exhaustive masking of motif...\n",
            "\t\tReprioritizing potential motifs...\n",
            "\t24 of 25 Initial Sequence: TAAATAAAAATA... (-10.012)\n",
            "\t\tRound 1: -6.43 TAAATAAAAATA T:11.0(1.09%),B:11.9(0.26%),P:1e-2\n",
            "\t\tRound 2: -6.43 TAAATAAAAATA T:11.0(1.09%),B:11.9(0.26%),P:1e-2\n",
            "\t\t=Final=: -6.50 TAAATAAAAATA T:11.0(1.10%),B:12.3(0.29%),P:1e-2\n",
            "\t\tPerforming exhaustive masking of motif...\n",
            "\t\tReprioritizing potential motifs...\n",
            "\t25 of 25 Initial Sequence: AGCAACAACTGG... (-10.012)\n",
            "\t\tRound 1: -16.08 AGCAACAACTGG T:12.0(1.19%),B:2.5(0.05%),P:1e-6\n",
            "\t\tRound 2: -17.81 AGCAACAACTGG T:17.0(1.69%),B:6.3(0.14%),P:1e-7\n",
            "\t\tRound 3: -17.81 AGCAACAACTGG T:17.0(1.69%),B:6.3(0.14%),P:1e-7\n",
            "\t\t=Final=: -18.08 AGCAACAACTGG T:17.0(1.70%),B:6.3(0.15%),P:1e-7\n",
            "\t\tPerforming exhaustive masking of motif...\n",
            "\t\tReprioritizing potential motifs...\n",
            "\n",
            "\tFinalizing Enrichment Statistics (new in v3.4)\n",
            "\tReading input files...\n",
            "\t5993 total sequences read\n",
            "\tCache length = 11180\n",
            "\tUsing hypergeometric scoring\n",
            "\tChecking enrichment of 25 motif(s)\n",
            "\t|0%                                    50%                                  100%|\n",
            "\t=================================================================================\n",
            "\tOutput in file: output_dir/homerMotifs.motifs12\n",
            "\n",
            "\t(Motifs in homer2 format)\n",
            "\tDetermining similar motifs...\t70 reduced to 18 motifs\n",
            "\tOutputing HTML and sequence logos for motif comparison...\n",
            "\tChecking de novo motifs against known motifs...\n",
            "\tFormatting HTML page...\n",
            "\t\t1 of 18 (1e-301) similar to SPT23/SPT23_YPD/[](Harbison)/Yeast(0.829)\n",
            "\t\t2 of 18 (1e-206) similar to hINR(CPE)(0.642)\n",
            "\t\t3 of 18 (1e-165) similar to pan/dmmpmm(Bigfoot)/fly(0.847)\n",
            "\t\t4 of 18 (1e-164) similar to BHLHE22/MA1635.2/Jaspar(0.748)\n",
            "\t\t5 of 18 (1e-138) similar to ZIPIC/MA2322.1/Jaspar(0.764)\n",
            "\t\t6 of 18 (1e-111) similar to FEZF2/MA2341.1/Jaspar(0.784)\n",
            "\t\t7 of 18 (1e-102) similar to NKX2-2/MA1645.2/Jaspar(0.696)\n",
            "\t\t8 of 18 (1e-59) similar to CDC5/MA0579.2/Jaspar(0.769)\n",
            "\t\t9 of 18 (1e-50) similar to AT3G46070/MA1381.2/Jaspar(0.734)\n",
            "\t\t10 of 18 (1e-30) similar to slbo/dmmpmm(Down)/fly(0.702)\n",
            "\t\t11 of 18 (1e-30) similar to NHLH1/MA0048.3/Jaspar(0.695)\n",
            "\t\t12 of 18 (1e-24) similar to ROX1/Literature(Harbison)/Yeast(0.800)\n",
            "\t\t13 of 18 (1e-20) similar to GATA20/MA1324.1/Jaspar(0.816)\n",
            "\t\t14 of 18 (1e-13) similar to sens/MA2271.1/Jaspar(0.756)\n",
            "\t\t15 of 18 (1e-13) similar to RVE4/MA1187.2/Jaspar(0.761)\n",
            "\t\t16 of 18 (1e-11) similar to Aef1/dmmpmm(Pollard)/fly(0.773)\n",
            "\t\t17 of 18 (1e-10) similar to NAC079/MA0939.1/Jaspar(0.685)\n",
            "\t\t18 of 18 (1e-7) similar to GATA11/MA1014.1/Jaspar(0.708)\n",
            "\tJob finished\n",
            "\n"
          ]
        }
      ]
    },
    {
      "cell_type": "code",
      "source": [
        "from google.colab import drive\n",
        "drive.mount('/content/drive')"
      ],
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "qcYbL4314LUx",
        "outputId": "37379a92-dc61-4a4a-9d64-5e1c441b2192"
      },
      "execution_count": null,
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "Mounted at /content/drive\n"
          ]
        }
      ]
    }
  ]
}